--- correctOutput/array_hard.s5	2015-02-09 13:34:57.894172000 +0100
+++ testOutput/array_hard.s5	2015-02-13 13:27:20.065436000 +0100
@@ -1,7 +1,6 @@
  PROGRAM()()
  FUNCTION_LIST()()
   FUNCTION(VOID)("main")()
-   (nil)
    STATEMENT_LIST()()
     DECLARATION_STATEMENT(ARRAY(INT)[3][3])("a")()
     ASSIGNMENT_STATEMENT()()
@@ -57,7 +56,6 @@
        VARIABLE()("c")()
        CONSTANT(INT)(1)()
   FUNCTION(ARRAY(INT)[3])("create")()
-   (nil)
    STATEMENT_LIST()()
     DECLARATION_STATEMENT(ARRAY(INT)[3])("a")()
     ASSIGNMENT_STATEMENT()()
