--- correctOutput/array_hard.s5	2015-02-09 13:34:57.894172000 +0100
+++ testOutput/array_hard.s5	2015-02-13 14:27:28.895477000 +0100
@@ -1,6 +1,8 @@
  PROGRAM()()
  FUNCTION_LIST()()
   FUNCTION(VOID)("main")()
+   TYPE(VOID)()
+   VARIABLE()("main")()
    (nil)
    STATEMENT_LIST()()
     DECLARATION_STATEMENT(ARRAY(INT)[3][3])("a")()
@@ -57,6 +59,8 @@
        VARIABLE()("c")()
        CONSTANT(INT)(1)()
   FUNCTION(ARRAY(INT)[3])("create")()
+   TYPE(ARRAY(INT)[3])()
+   VARIABLE()("create")()
    (nil)
    STATEMENT_LIST()()
     DECLARATION_STATEMENT(ARRAY(INT)[3])("a")()
