--- correctOutput/fibonacci_iterative.s4	2015-02-09 13:34:57.946167000 +0100
+++ testOutput/fibonacci_iterative.s4	2015-02-13 13:53:42.696208000 +0100
@@ -1,93 +1,93 @@
  Simplify FUNCTION_LIST 
-  Simplify FUNCTION 
-   Simplify TYPE 
-   Simplify PARAMETER_LIST 
-    Simplify VARIABLE_LIST 
-     Simplify DECLARATION_STATEMENT 
-      Simplify TYPE 
-   Simplify STATEMENT_LIST 
+   Simplify FUNCTION 
+    Simplify TYPE 
+    Simplify PARAMETER_LIST 
+      Simplify VARIABLE_LIST 
+        Simplify DECLARATION_STATEMENT 
+          Simplify TYPE 
     Simplify STATEMENT_LIST 
-     Simplify STATEMENT_LIST 
       Simplify STATEMENT_LIST 
-       Simplify STATEMENT_LIST 
         Simplify STATEMENT_LIST 
-         Simplify STATEMENT_LIST 
           Simplify STATEMENT_LIST 
-           Simplify STATEMENT_LIST 
-            Simplify STATEMENT 
-             Simplify DECLARATION_STATEMENT 
-              Simplify TYPE 
-           Simplify STATEMENT 
-            Simplify DECLARATION_STATEMENT 
-             Simplify TYPE 
-          Simplify STATEMENT 
-           Simplify DECLARATION_STATEMENT 
-            Simplify TYPE 
-         Simplify STATEMENT 
-           Simplify EXPRESSION (VARIABLE) 
-           Simplify EXPRESSION (CONSTANT) 
-        Simplify STATEMENT 
-          Simplify EXPRESSION (VARIABLE) 
-          Simplify EXPRESSION (CONSTANT) 
-       Simplify STATEMENT 
-         Simplify EXPRESSION (VARIABLE) 
-         Simplify EXPRESSION (CONSTANT) 
-      Simplify STATEMENT 
-        Simplify EXPRESSION (>=) 
-         Simplify EXPRESSION () 
-          Simplify EXPRESSION (VARIABLE) 
-         Simplify EXPRESSION (CONSTANT) 
-        Simplify STATEMENT_LIST 
-         Simplify STATEMENT_LIST 
-          Simplify STATEMENT_LIST 
-           Simplify STATEMENT 
-            Simplify DECLARATION_STATEMENT 
-             Simplify TYPE 
-          Simplify STATEMENT 
-            Simplify EXPRESSION (VARIABLE) 
-            Simplify EXPRESSION (CONSTANT) 
-         Simplify STATEMENT 
-           Simplify EXPRESSION () 
-            Simplify EXPRESSION (<) 
-             Simplify EXPRESSION () 
-              Simplify EXPRESSION (VARIABLE) 
-             Simplify EXPRESSION () 
-              Simplify EXPRESSION (VARIABLE) 
-           Simplify STATEMENT_LIST 
             Simplify STATEMENT_LIST 
-             Simplify STATEMENT_LIST 
               Simplify STATEMENT_LIST 
-               Simplify STATEMENT 
-                 Simplify EXPRESSION (VARIABLE) 
-                 Simplify EXPRESSION (+) 
-                  Simplify EXPRESSION () 
-                   Simplify EXPRESSION (VARIABLE) 
-                  Simplify EXPRESSION () 
+                Simplify STATEMENT_LIST 
+                  Simplify STATEMENT_LIST 
+                    Simplify STATEMENT_LIST 
+                      Simplify STATEMENT 
+                        Simplify DECLARATION_STATEMENT 
+                          Simplify TYPE 
+                    Simplify STATEMENT 
+                      Simplify DECLARATION_STATEMENT 
+                        Simplify TYPE 
+                  Simplify STATEMENT 
+                    Simplify DECLARATION_STATEMENT 
+                      Simplify TYPE 
+                Simplify STATEMENT 
                    Simplify EXPRESSION (VARIABLE) 
+                   Simplify EXPRESSION (CONSTANT) 
               Simplify STATEMENT 
-                Simplify EXPRESSION (VARIABLE) 
-                Simplify EXPRESSION () 
                  Simplify EXPRESSION (VARIABLE) 
-             Simplify STATEMENT 
-               Simplify EXPRESSION (VARIABLE) 
-               Simplify EXPRESSION () 
-                Simplify EXPRESSION (VARIABLE) 
+                 Simplify EXPRESSION (CONSTANT) 
             Simplify STATEMENT 
-              Simplify EXPRESSION (VARIABLE) 
-              Simplify EXPRESSION (+) 
+               Simplify EXPRESSION (VARIABLE) 
+               Simplify EXPRESSION (CONSTANT) 
+          Simplify STATEMENT 
+             Simplify EXPRESSION (>=) 
                Simplify EXPRESSION () 
-                Simplify EXPRESSION (VARIABLE) 
+                 Simplify EXPRESSION (VARIABLE) 
+               Simplify EXPRESSION (CONSTANT) 
+             Simplify STATEMENT_LIST 
+               Simplify STATEMENT_LIST 
+                 Simplify STATEMENT_LIST 
+                   Simplify STATEMENT 
+                     Simplify DECLARATION_STATEMENT 
+                       Simplify TYPE 
+                 Simplify STATEMENT 
+                    Simplify EXPRESSION (VARIABLE) 
+                    Simplify EXPRESSION (CONSTANT) 
+               Simplify STATEMENT 
+                  Simplify EXPRESSION () 
+                    Simplify EXPRESSION (<) 
+                      Simplify EXPRESSION () 
+                        Simplify EXPRESSION (VARIABLE) 
+                      Simplify EXPRESSION () 
+                        Simplify EXPRESSION (VARIABLE) 
+                  Simplify STATEMENT_LIST 
+                    Simplify STATEMENT_LIST 
+                      Simplify STATEMENT_LIST 
+                        Simplify STATEMENT_LIST 
+                          Simplify STATEMENT 
+                             Simplify EXPRESSION (VARIABLE) 
+                             Simplify EXPRESSION (+) 
+                               Simplify EXPRESSION () 
+                                 Simplify EXPRESSION (VARIABLE) 
+                               Simplify EXPRESSION () 
+                                 Simplify EXPRESSION (VARIABLE) 
+                        Simplify STATEMENT 
+                           Simplify EXPRESSION (VARIABLE) 
+                           Simplify EXPRESSION () 
+                             Simplify EXPRESSION (VARIABLE) 
+                      Simplify STATEMENT 
+                         Simplify EXPRESSION (VARIABLE) 
+                         Simplify EXPRESSION () 
+                           Simplify EXPRESSION (VARIABLE) 
+                    Simplify STATEMENT 
+                       Simplify EXPRESSION (VARIABLE) 
+                       Simplify EXPRESSION (+) 
+                         Simplify EXPRESSION () 
+                           Simplify EXPRESSION (VARIABLE) 
+                         Simplify EXPRESSION (CONSTANT) 
+        Simplify STATEMENT 
+           Simplify EXPRESSION_LIST 
+             Simplify EXPRESSION_LIST 
+               Simplify EXPRESSION_LIST 
+                 Simplify EXPRESSION_LIST 
+                   Simplify EXPRESSION (CONSTANT) 
+                 Simplify EXPRESSION () 
+                   Simplify EXPRESSION (VARIABLE) 
                Simplify EXPRESSION (CONSTANT) 
-     Simplify STATEMENT 
-       Simplify EXPRESSION_LIST 
-        Simplify EXPRESSION_LIST 
-         Simplify EXPRESSION_LIST 
-          Simplify EXPRESSION_LIST 
-           Simplify EXPRESSION (CONSTANT) 
-          Simplify EXPRESSION () 
-           Simplify EXPRESSION (VARIABLE) 
+             Simplify EXPRESSION () 
+               Simplify EXPRESSION (VARIABLE) 
+      Simplify STATEMENT 
          Simplify EXPRESSION (CONSTANT) 
-        Simplify EXPRESSION () 
-         Simplify EXPRESSION (VARIABLE) 
-    Simplify STATEMENT 
-      Simplify EXPRESSION (CONSTANT) 
