--- correctOutput/fibonacci_iterative.s8	2015-02-09 13:34:57.949170000 +0100
+++ testOutput/fibonacci_iterative.s8	2015-02-13 16:38:54.637206000 +0100
@@ -8,9 +8,23 @@
 .INTEGER: .ascii "%d \000"
 .FLOAT: .ascii "%f \000"
 .NEWLINE: .ascii "\n \000"
-.STRING0: .ascii "Fibonacci number #"
+.STRING0: .ascii (null)
 .ascii "\000"
-.STRING1: .ascii "is "
+.STRING1: .ascii (null)
+.ascii "\000"
+.STRING2: .ascii (null)
+.ascii "\000"
+.STRING3: .ascii (null)
+.ascii "\000"
+.STRING4: .ascii (null)
+.ascii "\000"
+.STRING5: .ascii (null)
+.ascii "\000"
+.STRING6: .ascii (null)
+.ascii "\000"
+.STRING7: .ascii (null)
+.ascii "\000"
+.STRING8: .ascii (null)
 .ascii "\000"
 .globl main
 .align	2
