--- correctOutput/function_simple.s5	2015-02-09 13:34:58.007171000 +0100
+++ testOutput/function_simple.s5	2015-02-13 14:27:30.706487000 +0100
@@ -1,6 +1,8 @@
  PROGRAM()()
  FUNCTION_LIST()()
   FUNCTION(VOID)("main")()
+   TYPE(VOID)()
+   VARIABLE()("main")()
    (nil)
    STATEMENT_LIST()()
     PRINT_STATEMENT()()
@@ -10,6 +12,8 @@
      VARIABLE()("function")()
      (nil)
   FUNCTION(VOID)("function")()
+   TYPE(VOID)()
+   VARIABLE()("function")()
    (nil)
    STATEMENT_LIST()()
     PRINT_STATEMENT()()
