--- correctOutput/function_hard.s5	2015-02-09 13:34:57.987172000 +0100
+++ testOutput/function_hard.s5	2015-02-12 15:48:53.726113000 +0100
@@ -1,7 +1,6 @@
  PROGRAM()()
  FUNCTION_LIST()()
   FUNCTION(VOID)("main")()
-   (nil)
    STATEMENT_LIST()()
     DECLARATION_STATEMENT(INT)("a")()
     DECLARATION_STATEMENT(INT)("b")()
@@ -23,6 +22,7 @@
         VARIABLE()("a")()
         VARIABLE()("b")()
         VARIABLE()("c")()
+  (nil)
   FUNCTION(INT)("func1")()
    VARIABLE_LIST()()
     DECLARATION_STATEMENT(INT)("a")()
