--- correctOutput/hello.s4	2015-02-09 13:34:58.017175000 +0100
+++ testOutput/hello.s4	2015-02-13 13:53:46.295228000 +0100
@@ -1,7 +1,7 @@
  Simplify FUNCTION_LIST 
-  Simplify FUNCTION 
-   Simplify TYPE 
-   Simplify STATEMENT_LIST 
-    Simplify STATEMENT 
-      Simplify EXPRESSION_LIST 
-       Simplify EXPRESSION (CONSTANT) 
+   Simplify FUNCTION 
+    Simplify TYPE 
+    Simplify STATEMENT_LIST 
+      Simplify STATEMENT 
+         Simplify EXPRESSION_LIST 
+           Simplify EXPRESSION (CONSTANT) 
