--- correctOutput/if_simple.s4	2015-02-09 13:34:58.037173000 +0100
+++ testOutput/if_simple.s4	2015-02-13 13:53:47.044233000 +0100
@@ -1,17 +1,17 @@
  Simplify FUNCTION_LIST 
-  Simplify FUNCTION 
-   Simplify TYPE 
-   Simplify STATEMENT_LIST 
+   Simplify FUNCTION 
+    Simplify TYPE 
     Simplify STATEMENT_LIST 
-     Simplify STATEMENT 
-       Simplify EXPRESSION (CONSTANT) 
-       Simplify STATEMENT_LIST 
+      Simplify STATEMENT_LIST 
         Simplify STATEMENT 
-          Simplify EXPRESSION_LIST 
            Simplify EXPRESSION (CONSTANT) 
-    Simplify STATEMENT 
-      Simplify EXPRESSION (CONSTANT) 
-      Simplify STATEMENT_LIST 
-       Simplify STATEMENT 
-         Simplify EXPRESSION_LIST 
-          Simplify EXPRESSION (CONSTANT) 
+           Simplify STATEMENT_LIST 
+             Simplify STATEMENT 
+                Simplify EXPRESSION_LIST 
+                  Simplify EXPRESSION (CONSTANT) 
+      Simplify STATEMENT 
+         Simplify EXPRESSION (CONSTANT) 
+         Simplify STATEMENT_LIST 
+           Simplify STATEMENT 
+              Simplify EXPRESSION_LIST 
+                Simplify EXPRESSION (CONSTANT) 
