--- correctOutput/if_simple.s8	2015-02-09 13:34:58.048168000 +0100
+++ testOutput/if_simple.s8	2015-02-13 16:38:55.760213000 +0100
@@ -8,9 +8,13 @@
 .INTEGER: .ascii "%d \000"
 .FLOAT: .ascii "%f \000"
 .NEWLINE: .ascii "\n \000"
-.STRING0: .ascii "true"
+.STRING0: .ascii (null)
 .ascii "\000"
-.STRING1: .ascii "false"
+.STRING1: .ascii (null)
+.ascii "\000"
+.STRING2: .ascii (null)
+.ascii "\000"
+.STRING3: .ascii (null)
 .ascii "\000"
 .globl main
 .align	2
