--- correctOutput/logical.s4	2015-02-09 13:34:58.054168000 +0100
+++ testOutput/logical.s4	2015-02-13 13:56:47.585242000 +0100
@@ -2,303 +2,303 @@
   Simplify FUNCTION 
    Simplify TYPE 
    Simplify STATEMENT_LIST 
-    Simplify STATEMENT_LIST 
      Simplify STATEMENT_LIST 
-      Simplify STATEMENT_LIST 
        Simplify STATEMENT_LIST 
-        Simplify STATEMENT_LIST 
          Simplify STATEMENT_LIST 
-          Simplify STATEMENT_LIST 
            Simplify STATEMENT_LIST 
-            Simplify STATEMENT_LIST 
              Simplify STATEMENT_LIST 
-              Simplify STATEMENT_LIST 
                Simplify STATEMENT_LIST 
-                Simplify STATEMENT_LIST 
                  Simplify STATEMENT_LIST 
-                  Simplify STATEMENT_LIST 
                    Simplify STATEMENT_LIST 
-                    Simplify STATEMENT_LIST 
                      Simplify STATEMENT_LIST 
-                      Simplify STATEMENT_LIST 
                        Simplify STATEMENT_LIST 
-                        Simplify STATEMENT_LIST 
                          Simplify STATEMENT_LIST 
-                          Simplify STATEMENT_LIST 
                            Simplify STATEMENT_LIST 
-                            Simplify STATEMENT_LIST 
                              Simplify STATEMENT_LIST 
-                              Simplify STATEMENT_LIST 
                                Simplify STATEMENT_LIST 
-                                Simplify STATEMENT_LIST 
                                  Simplify STATEMENT_LIST 
-                                  Simplify STATEMENT_LIST 
                                    Simplify STATEMENT_LIST 
-                                    Simplify STATEMENT_LIST 
                                      Simplify STATEMENT_LIST 
-                                      Simplify STATEMENT_LIST 
                                        Simplify STATEMENT_LIST 
-                                        Simplify STATEMENT 
-                                         Simplify DECLARATION_STATEMENT 
-                                          Simplify TYPE 
+                                         Simplify STATEMENT_LIST 
+                                           Simplify STATEMENT_LIST 
+                                             Simplify STATEMENT_LIST 
+                                               Simplify STATEMENT_LIST 
+                                                 Simplify STATEMENT_LIST 
+                                                   Simplify STATEMENT_LIST 
+                                                     Simplify STATEMENT_LIST 
+                                                       Simplify STATEMENT_LIST 
+                                                         Simplify STATEMENT_LIST 
+                                                           Simplify STATEMENT_LIST 
+                                                             Simplify STATEMENT_LIST 
+                                                               Simplify STATEMENT_LIST 
+                                                                 Simplify STATEMENT_LIST 
+                                                                   Simplify STATEMENT_LIST 
+                                                                     Simplify STATEMENT_LIST 
+                                                                       Simplify STATEMENT_LIST 
+                                                                         Simplify STATEMENT_LIST 
+                                                                           Simplify STATEMENT_LIST 
+                                                                             Simplify STATEMENT 
+                                                                               Simplify DECLARATION_STATEMENT 
+                                                                                 Simplify TYPE 
+                                                                           Simplify STATEMENT 
+                                                                             Simplify DECLARATION_STATEMENT 
+                                                                               Simplify TYPE 
+                                                                         Simplify STATEMENT 
+                                                                           Simplify DECLARATION_STATEMENT 
+                                                                             Simplify TYPE 
+                                                                       Simplify STATEMENT 
+                                                                         Simplify DECLARATION_STATEMENT 
+                                                                           Simplify TYPE 
+                                                                     Simplify STATEMENT 
+                                                                       Simplify DECLARATION_STATEMENT 
+                                                                         Simplify TYPE 
+                                                                   Simplify STATEMENT 
+                                                                     Simplify DECLARATION_STATEMENT 
+                                                                       Simplify TYPE 
+                                                                 Simplify STATEMENT 
+                                                                    Simplify EXPRESSION (VARIABLE) 
+                                                                    Simplify EXPRESSION (CONSTANT) 
+                                                               Simplify STATEMENT 
+                                                                  Simplify EXPRESSION (VARIABLE) 
+                                                                  Simplify EXPRESSION (CONSTANT) 
+                                                             Simplify STATEMENT 
+                                                                Simplify EXPRESSION (VARIABLE) 
+                                                                Simplify EXPRESSION (CONSTANT) 
+                                                           Simplify STATEMENT 
+                                                              Simplify EXPRESSION (VARIABLE) 
+                                                              Simplify EXPRESSION (&&) 
+                                                                Simplify EXPRESSION () 
+                                                                  Simplify EXPRESSION (VARIABLE) 
+                                                                Simplify EXPRESSION () 
+                                                                  Simplify EXPRESSION (VARIABLE) 
+                                                         Simplify STATEMENT 
+                                                            Simplify EXPRESSION (VARIABLE) 
+                                                            Simplify EXPRESSION (&&) 
+                                                              Simplify EXPRESSION () 
+                                                                Simplify EXPRESSION (VARIABLE) 
+                                                              Simplify EXPRESSION () 
+                                                                Simplify EXPRESSION (VARIABLE) 
+                                                       Simplify STATEMENT 
+                                                          Simplify EXPRESSION (VARIABLE) 
+                                                          Simplify EXPRESSION (&&) 
+                                                            Simplify EXPRESSION () 
+                                                              Simplify EXPRESSION (VARIABLE) 
+                                                            Simplify EXPRESSION () 
+                                                              Simplify EXPRESSION (VARIABLE) 
+                                                     Simplify STATEMENT 
+                                                        Simplify EXPRESSION_LIST 
+                                                          Simplify EXPRESSION_LIST 
+                                                            Simplify EXPRESSION_LIST 
+                                                              Simplify EXPRESSION () 
+                                                                Simplify EXPRESSION (VARIABLE) 
+                                                            Simplify EXPRESSION () 
+                                                              Simplify EXPRESSION (VARIABLE) 
+                                                          Simplify EXPRESSION () 
+                                                            Simplify EXPRESSION (VARIABLE) 
+                                                   Simplify STATEMENT 
+                                                      Simplify EXPRESSION (VARIABLE) 
+                                                      Simplify EXPRESSION (||) 
+                                                        Simplify EXPRESSION () 
+                                                          Simplify EXPRESSION (VARIABLE) 
+                                                        Simplify EXPRESSION () 
+                                                          Simplify EXPRESSION (VARIABLE) 
+                                                 Simplify STATEMENT 
+                                                    Simplify EXPRESSION (VARIABLE) 
+                                                    Simplify EXPRESSION (||) 
+                                                      Simplify EXPRESSION () 
+                                                        Simplify EXPRESSION (VARIABLE) 
+                                                      Simplify EXPRESSION () 
+                                                        Simplify EXPRESSION (VARIABLE) 
+                                               Simplify STATEMENT 
+                                                  Simplify EXPRESSION (VARIABLE) 
+                                                  Simplify EXPRESSION (||) 
+                                                    Simplify EXPRESSION () 
+                                                      Simplify EXPRESSION (VARIABLE) 
+                                                    Simplify EXPRESSION () 
+                                                      Simplify EXPRESSION (VARIABLE) 
+                                             Simplify STATEMENT 
+                                                Simplify EXPRESSION_LIST 
+                                                  Simplify EXPRESSION_LIST 
+                                                    Simplify EXPRESSION_LIST 
+                                                      Simplify EXPRESSION () 
+                                                        Simplify EXPRESSION (VARIABLE) 
+                                                    Simplify EXPRESSION () 
+                                                      Simplify EXPRESSION (VARIABLE) 
+                                                  Simplify EXPRESSION () 
+                                                    Simplify EXPRESSION (VARIABLE) 
+                                           Simplify STATEMENT 
+                                              Simplify EXPRESSION (VARIABLE) 
+                                              Simplify EXPRESSION (!) 
+                                                Simplify EXPRESSION () 
+                                                  Simplify EXPRESSION (VARIABLE) 
+                                         Simplify STATEMENT 
+                                            Simplify EXPRESSION (VARIABLE) 
+                                            Simplify EXPRESSION (!) 
+                                              Simplify EXPRESSION () 
+                                                Simplify EXPRESSION (VARIABLE) 
                                        Simplify STATEMENT 
-                                        Simplify DECLARATION_STATEMENT 
-                                         Simplify TYPE 
-                                      Simplify STATEMENT 
-                                       Simplify DECLARATION_STATEMENT 
-                                        Simplify TYPE 
+                                          Simplify EXPRESSION (VARIABLE) 
+                                          Simplify EXPRESSION (!) 
+                                            Simplify EXPRESSION () 
+                                              Simplify EXPRESSION (VARIABLE) 
                                      Simplify STATEMENT 
-                                      Simplify DECLARATION_STATEMENT 
-                                       Simplify TYPE 
-                                    Simplify STATEMENT 
-                                     Simplify DECLARATION_STATEMENT 
-                                      Simplify TYPE 
+                                        Simplify EXPRESSION_LIST 
+                                          Simplify EXPRESSION_LIST 
+                                            Simplify EXPRESSION_LIST 
+                                              Simplify EXPRESSION () 
+                                                Simplify EXPRESSION (VARIABLE) 
+                                            Simplify EXPRESSION () 
+                                              Simplify EXPRESSION (VARIABLE) 
+                                          Simplify EXPRESSION () 
+                                            Simplify EXPRESSION (VARIABLE) 
                                    Simplify STATEMENT 
-                                    Simplify DECLARATION_STATEMENT 
-                                     Simplify TYPE 
-                                  Simplify STATEMENT 
-                                    Simplify EXPRESSION (VARIABLE) 
-                                    Simplify EXPRESSION (CONSTANT) 
+                                      Simplify EXPRESSION (VARIABLE) 
+                                      Simplify EXPRESSION (&&) 
+                                        Simplify EXPRESSION () 
+                                          Simplify EXPRESSION (VARIABLE) 
+                                        Simplify EXPRESSION (!) 
+                                          Simplify EXPRESSION () 
+                                            Simplify EXPRESSION (VARIABLE) 
                                  Simplify STATEMENT 
-                                   Simplify EXPRESSION (VARIABLE) 
-                                   Simplify EXPRESSION (CONSTANT) 
-                                Simplify STATEMENT 
-                                  Simplify EXPRESSION (VARIABLE) 
-                                  Simplify EXPRESSION (CONSTANT) 
+                                    Simplify EXPRESSION (VARIABLE) 
+                                    Simplify EXPRESSION (&&) 
+                                      Simplify EXPRESSION () 
+                                        Simplify EXPRESSION (VARIABLE) 
+                                      Simplify EXPRESSION (!) 
+                                        Simplify EXPRESSION () 
+                                          Simplify EXPRESSION (VARIABLE) 
                                Simplify STATEMENT 
-                                 Simplify EXPRESSION (VARIABLE) 
-                                 Simplify EXPRESSION (&&) 
-                                  Simplify EXPRESSION () 
-                                   Simplify EXPRESSION (VARIABLE) 
-                                  Simplify EXPRESSION () 
-                                   Simplify EXPRESSION (VARIABLE) 
-                              Simplify STATEMENT 
-                                Simplify EXPRESSION (VARIABLE) 
-                                Simplify EXPRESSION (&&) 
-                                 Simplify EXPRESSION () 
-                                  Simplify EXPRESSION (VARIABLE) 
-                                 Simplify EXPRESSION () 
                                   Simplify EXPRESSION (VARIABLE) 
+                                  Simplify EXPRESSION (&&) 
+                                    Simplify EXPRESSION () 
+                                      Simplify EXPRESSION (VARIABLE) 
+                                    Simplify EXPRESSION (!) 
+                                      Simplify EXPRESSION () 
+                                        Simplify EXPRESSION (VARIABLE) 
                              Simplify STATEMENT 
-                               Simplify EXPRESSION (VARIABLE) 
-                               Simplify EXPRESSION (&&) 
-                                Simplify EXPRESSION () 
-                                 Simplify EXPRESSION (VARIABLE) 
-                                Simplify EXPRESSION () 
-                                 Simplify EXPRESSION (VARIABLE) 
-                            Simplify STATEMENT 
-                              Simplify EXPRESSION_LIST 
-                               Simplify EXPRESSION_LIST 
                                 Simplify EXPRESSION_LIST 
-                                 Simplify EXPRESSION () 
+                                  Simplify EXPRESSION_LIST 
+                                    Simplify EXPRESSION_LIST 
+                                      Simplify EXPRESSION () 
+                                        Simplify EXPRESSION (VARIABLE) 
+                                    Simplify EXPRESSION () 
+                                      Simplify EXPRESSION (VARIABLE) 
+                                  Simplify EXPRESSION () 
+                                    Simplify EXPRESSION (VARIABLE) 
+                           Simplify STATEMENT 
+                              Simplify EXPRESSION (VARIABLE) 
+                              Simplify EXPRESSION (||) 
+                                Simplify EXPRESSION (!) 
+                                  Simplify EXPRESSION () 
+                                    Simplify EXPRESSION (VARIABLE) 
+                                Simplify EXPRESSION () 
                                   Simplify EXPRESSION (VARIABLE) 
+                         Simplify STATEMENT 
+                            Simplify EXPRESSION (VARIABLE) 
+                            Simplify EXPRESSION (||) 
+                              Simplify EXPRESSION (!) 
                                 Simplify EXPRESSION () 
-                                 Simplify EXPRESSION (VARIABLE) 
-                               Simplify EXPRESSION () 
-                                Simplify EXPRESSION (VARIABLE) 
-                           Simplify STATEMENT 
-                             Simplify EXPRESSION (VARIABLE) 
-                             Simplify EXPRESSION (||) 
+                                  Simplify EXPRESSION (VARIABLE) 
                               Simplify EXPRESSION () 
-                               Simplify EXPRESSION (VARIABLE) 
+                                Simplify EXPRESSION (VARIABLE) 
+                       Simplify STATEMENT 
+                          Simplify EXPRESSION (VARIABLE) 
+                          Simplify EXPRESSION (||) 
+                            Simplify EXPRESSION (!) 
                               Simplify EXPRESSION () 
-                               Simplify EXPRESSION (VARIABLE) 
-                          Simplify STATEMENT 
-                            Simplify EXPRESSION (VARIABLE) 
-                            Simplify EXPRESSION (||) 
-                             Simplify EXPRESSION () 
-                              Simplify EXPRESSION (VARIABLE) 
-                             Simplify EXPRESSION () 
-                              Simplify EXPRESSION (VARIABLE) 
-                         Simplify STATEMENT 
-                           Simplify EXPRESSION (VARIABLE) 
-                           Simplify EXPRESSION (||) 
-                            Simplify EXPRESSION () 
-                             Simplify EXPRESSION (VARIABLE) 
+                                Simplify EXPRESSION (VARIABLE) 
                             Simplify EXPRESSION () 
-                             Simplify EXPRESSION (VARIABLE) 
-                        Simplify STATEMENT 
+                              Simplify EXPRESSION (VARIABLE) 
+                     Simplify STATEMENT 
+                        Simplify EXPRESSION_LIST 
                           Simplify EXPRESSION_LIST 
-                           Simplify EXPRESSION_LIST 
                             Simplify EXPRESSION_LIST 
-                             Simplify EXPRESSION () 
-                              Simplify EXPRESSION (VARIABLE) 
+                              Simplify EXPRESSION () 
+                                Simplify EXPRESSION (VARIABLE) 
                             Simplify EXPRESSION () 
-                             Simplify EXPRESSION (VARIABLE) 
-                           Simplify EXPRESSION () 
+                              Simplify EXPRESSION (VARIABLE) 
+                          Simplify EXPRESSION () 
                             Simplify EXPRESSION (VARIABLE) 
-                       Simplify STATEMENT 
-                         Simplify EXPRESSION (VARIABLE) 
-                         Simplify EXPRESSION (!) 
+                   Simplify STATEMENT 
+                      Simplify EXPRESSION (VARIABLE) 
+                      Simplify EXPRESSION (&&) 
+                        Simplify EXPRESSION (&&) 
                           Simplify EXPRESSION () 
-                           Simplify EXPRESSION (VARIABLE) 
-                      Simplify STATEMENT 
-                        Simplify EXPRESSION (VARIABLE) 
-                        Simplify EXPRESSION (!) 
-                         Simplify EXPRESSION () 
+                            Simplify EXPRESSION (VARIABLE) 
+                          Simplify EXPRESSION () 
+                            Simplify EXPRESSION (VARIABLE) 
+                        Simplify EXPRESSION () 
                           Simplify EXPRESSION (VARIABLE) 
-                     Simplify STATEMENT 
-                       Simplify EXPRESSION (VARIABLE) 
-                       Simplify EXPRESSION (!) 
+                 Simplify STATEMENT 
+                    Simplify EXPRESSION (VARIABLE) 
+                    Simplify EXPRESSION (||) 
+                      Simplify EXPRESSION (||) 
                         Simplify EXPRESSION () 
-                         Simplify EXPRESSION (VARIABLE) 
-                    Simplify STATEMENT 
-                      Simplify EXPRESSION_LIST 
-                       Simplify EXPRESSION_LIST 
-                        Simplify EXPRESSION_LIST 
-                         Simplify EXPRESSION () 
                           Simplify EXPRESSION (VARIABLE) 
                         Simplify EXPRESSION () 
-                         Simplify EXPRESSION (VARIABLE) 
-                       Simplify EXPRESSION () 
-                        Simplify EXPRESSION (VARIABLE) 
-                   Simplify STATEMENT 
-                     Simplify EXPRESSION (VARIABLE) 
-                     Simplify EXPRESSION (&&) 
+                          Simplify EXPRESSION (VARIABLE) 
                       Simplify EXPRESSION () 
-                       Simplify EXPRESSION (VARIABLE) 
-                      Simplify EXPRESSION (!) 
-                       Simplify EXPRESSION () 
                         Simplify EXPRESSION (VARIABLE) 
-                  Simplify STATEMENT 
-                    Simplify EXPRESSION (VARIABLE) 
+               Simplify STATEMENT 
+                  Simplify EXPRESSION (VARIABLE) 
+                  Simplify EXPRESSION (||) 
                     Simplify EXPRESSION (&&) 
-                     Simplify EXPRESSION () 
-                      Simplify EXPRESSION (VARIABLE) 
-                     Simplify EXPRESSION (!) 
                       Simplify EXPRESSION () 
-                       Simplify EXPRESSION (VARIABLE) 
-                 Simplify STATEMENT 
-                   Simplify EXPRESSION (VARIABLE) 
-                   Simplify EXPRESSION (&&) 
+                        Simplify EXPRESSION (VARIABLE) 
+                      Simplify EXPRESSION () 
+                        Simplify EXPRESSION (VARIABLE) 
                     Simplify EXPRESSION () 
-                     Simplify EXPRESSION (VARIABLE) 
-                    Simplify EXPRESSION (!) 
-                     Simplify EXPRESSION () 
                       Simplify EXPRESSION (VARIABLE) 
-                Simplify STATEMENT 
+             Simplify STATEMENT 
+                Simplify EXPRESSION_LIST 
                   Simplify EXPRESSION_LIST 
-                   Simplify EXPRESSION_LIST 
                     Simplify EXPRESSION_LIST 
-                     Simplify EXPRESSION () 
-                      Simplify EXPRESSION (VARIABLE) 
+                      Simplify EXPRESSION () 
+                        Simplify EXPRESSION (VARIABLE) 
                     Simplify EXPRESSION () 
-                     Simplify EXPRESSION (VARIABLE) 
-                   Simplify EXPRESSION () 
-                    Simplify EXPRESSION (VARIABLE) 
-               Simplify STATEMENT 
-                 Simplify EXPRESSION (VARIABLE) 
-                 Simplify EXPRESSION (||) 
-                  Simplify EXPRESSION (!) 
-                   Simplify EXPRESSION () 
-                    Simplify EXPRESSION (VARIABLE) 
-                  Simplify EXPRESSION () 
-                   Simplify EXPRESSION (VARIABLE) 
-              Simplify STATEMENT 
-                Simplify EXPRESSION (VARIABLE) 
-                Simplify EXPRESSION (||) 
-                 Simplify EXPRESSION (!) 
+                      Simplify EXPRESSION (VARIABLE) 
                   Simplify EXPRESSION () 
-                   Simplify EXPRESSION (VARIABLE) 
-                 Simplify EXPRESSION () 
-                  Simplify EXPRESSION (VARIABLE) 
-             Simplify STATEMENT 
-               Simplify EXPRESSION (VARIABLE) 
-               Simplify EXPRESSION (||) 
-                Simplify EXPRESSION (!) 
-                 Simplify EXPRESSION () 
-                  Simplify EXPRESSION (VARIABLE) 
+                    Simplify EXPRESSION (VARIABLE) 
+           Simplify STATEMENT 
+              Simplify EXPRESSION (VARIABLE) 
+              Simplify EXPRESSION (||) 
                 Simplify EXPRESSION () 
-                 Simplify EXPRESSION (VARIABLE) 
-            Simplify STATEMENT 
-              Simplify EXPRESSION_LIST 
-               Simplify EXPRESSION_LIST 
-                Simplify EXPRESSION_LIST 
-                 Simplify EXPRESSION () 
                   Simplify EXPRESSION (VARIABLE) 
-                Simplify EXPRESSION () 
-                 Simplify EXPRESSION (VARIABLE) 
-               Simplify EXPRESSION () 
-                Simplify EXPRESSION (VARIABLE) 
-           Simplify STATEMENT 
-             Simplify EXPRESSION (VARIABLE) 
-             Simplify EXPRESSION (&&) 
-              Simplify EXPRESSION (&&) 
-               Simplify EXPRESSION () 
-                Simplify EXPRESSION (VARIABLE) 
-               Simplify EXPRESSION () 
-                Simplify EXPRESSION (VARIABLE) 
-              Simplify EXPRESSION () 
-               Simplify EXPRESSION (VARIABLE) 
-          Simplify STATEMENT 
+                Simplify EXPRESSION (&&) 
+                  Simplify EXPRESSION () 
+                    Simplify EXPRESSION (VARIABLE) 
+                  Simplify EXPRESSION () 
+                    Simplify EXPRESSION (VARIABLE) 
+         Simplify STATEMENT 
             Simplify EXPRESSION (VARIABLE) 
-            Simplify EXPRESSION (||) 
-             Simplify EXPRESSION (||) 
+            Simplify EXPRESSION (&&) 
               Simplify EXPRESSION () 
-               Simplify EXPRESSION (VARIABLE) 
+                Simplify EXPRESSION (||) 
+                  Simplify EXPRESSION () 
+                    Simplify EXPRESSION (VARIABLE) 
+                  Simplify EXPRESSION () 
+                    Simplify EXPRESSION (VARIABLE) 
               Simplify EXPRESSION () 
-               Simplify EXPRESSION (VARIABLE) 
-             Simplify EXPRESSION () 
+                Simplify EXPRESSION (VARIABLE) 
+       Simplify STATEMENT 
+          Simplify EXPRESSION (VARIABLE) 
+          Simplify EXPRESSION (||) 
+            Simplify EXPRESSION () 
               Simplify EXPRESSION (VARIABLE) 
-         Simplify STATEMENT 
-           Simplify EXPRESSION (VARIABLE) 
-           Simplify EXPRESSION (||) 
             Simplify EXPRESSION (&&) 
-             Simplify EXPRESSION () 
-              Simplify EXPRESSION (VARIABLE) 
-             Simplify EXPRESSION () 
-              Simplify EXPRESSION (VARIABLE) 
-            Simplify EXPRESSION () 
-             Simplify EXPRESSION (VARIABLE) 
-        Simplify STATEMENT 
+              Simplify EXPRESSION () 
+                Simplify EXPRESSION (VARIABLE) 
+              Simplify EXPRESSION (!) 
+                Simplify EXPRESSION () 
+                  Simplify EXPRESSION (VARIABLE) 
+     Simplify STATEMENT 
+        Simplify EXPRESSION_LIST 
           Simplify EXPRESSION_LIST 
-           Simplify EXPRESSION_LIST 
             Simplify EXPRESSION_LIST 
-             Simplify EXPRESSION () 
-              Simplify EXPRESSION (VARIABLE) 
+              Simplify EXPRESSION () 
+                Simplify EXPRESSION (VARIABLE) 
             Simplify EXPRESSION () 
-             Simplify EXPRESSION (VARIABLE) 
-           Simplify EXPRESSION () 
-            Simplify EXPRESSION (VARIABLE) 
-       Simplify STATEMENT 
-         Simplify EXPRESSION (VARIABLE) 
-         Simplify EXPRESSION (||) 
+              Simplify EXPRESSION (VARIABLE) 
           Simplify EXPRESSION () 
-           Simplify EXPRESSION (VARIABLE) 
-          Simplify EXPRESSION (&&) 
-           Simplify EXPRESSION () 
-            Simplify EXPRESSION (VARIABLE) 
-           Simplify EXPRESSION () 
-            Simplify EXPRESSION (VARIABLE) 
-      Simplify STATEMENT 
-        Simplify EXPRESSION (VARIABLE) 
-        Simplify EXPRESSION (&&) 
-         Simplify EXPRESSION () 
-          Simplify EXPRESSION (||) 
-           Simplify EXPRESSION () 
-            Simplify EXPRESSION (VARIABLE) 
-           Simplify EXPRESSION () 
             Simplify EXPRESSION (VARIABLE) 
-         Simplify EXPRESSION () 
-          Simplify EXPRESSION (VARIABLE) 
-     Simplify STATEMENT 
-       Simplify EXPRESSION (VARIABLE) 
-       Simplify EXPRESSION (||) 
-        Simplify EXPRESSION () 
-         Simplify EXPRESSION (VARIABLE) 
-        Simplify EXPRESSION (&&) 
-         Simplify EXPRESSION () 
-          Simplify EXPRESSION (VARIABLE) 
-         Simplify EXPRESSION (!) 
-          Simplify EXPRESSION () 
-           Simplify EXPRESSION (VARIABLE) 
-    Simplify STATEMENT 
-      Simplify EXPRESSION_LIST 
-       Simplify EXPRESSION_LIST 
-        Simplify EXPRESSION_LIST 
-         Simplify EXPRESSION () 
-          Simplify EXPRESSION (VARIABLE) 
-        Simplify EXPRESSION () 
-         Simplify EXPRESSION (VARIABLE) 
-       Simplify EXPRESSION () 
-        Simplify EXPRESSION (VARIABLE) 
