--- correctOutput/maxmin.s5	2015-02-09 13:34:58.086168000 +0100
+++ testOutput/maxmin.s5	2015-02-13 14:27:31.585494000 +0100
@@ -1,6 +1,8 @@
  PROGRAM()()
  FUNCTION_LIST()()
   FUNCTION(VOID)("maxmin")()
+   TYPE(VOID)()
+   VARIABLE()("maxmin")()
    (nil)
    STATEMENT_LIST()()
     DECLARATION_STATEMENT(INT)("a")()
@@ -39,6 +41,8 @@
       CONSTANT(STRING)("c is: ")()
       VARIABLE()("c")()
   FUNCTION(INT)("max")()
+   TYPE(INT)()
+   VARIABLE()("max")()
    VARIABLE_LIST()()
     DECLARATION_STATEMENT(INT)("a")()
     DECLARATION_STATEMENT(INT)("b")()
@@ -60,6 +64,8 @@
     RETURN_STATEMENT()()
      CONSTANT(INT)(0)()
   FUNCTION(INT)("min")()
+   TYPE(INT)()
+   VARIABLE()("min")()
    VARIABLE_LIST()()
     DECLARATION_STATEMENT(INT)("a")()
     DECLARATION_STATEMENT(INT)("b")()
