--- correctOutput/merge_sort.s8	2015-02-09 13:34:58.099169000 +0100
+++ testOutput/merge_sort.s8	2015-02-13 16:38:56.341219000 +0100
@@ -8,5 +8,33 @@
 .INTEGER: .ascii "%d \000"
 .FLOAT: .ascii "%f \000"
 .NEWLINE: .ascii "\n \000"
+.STRING0: .ascii (null)
+.ascii "\000"
+.STRING1: .ascii (null)
+.ascii "\000"
+.STRING2: .ascii (null)
+.ascii "\000"
+.STRING3: .ascii (null)
+.ascii "\000"
+.STRING4: .ascii (null)
+.ascii "\000"
+.STRING5: .ascii (null)
+.ascii "\000"
+.STRING6: .ascii (null)
+.ascii "\000"
+.STRING7: .ascii (null)
+.ascii "\000"
+.STRING8: .ascii (null)
+.ascii "\000"
+.STRING9: .ascii (null)
+.ascii "\000"
+.STRING10: .ascii (null)
+.ascii "\000"
+.STRING11: .ascii (null)
+.ascii "\000"
+.STRING12: .ascii (null)
+.ascii "\000"
+.STRING13: .ascii (null)
+.ascii "\000"
 .globl main
 .align	2
