--- correctOutput/sorting.s5	2015-02-09 13:34:58.158174000 +0100
+++ testOutput/sorting.s5	2015-02-12 15:49:04.348171000 +0100
@@ -1,7 +1,6 @@
  PROGRAM()()
  FUNCTION_LIST()()
   FUNCTION(VOID)("main")()
-   (nil)
    STATEMENT_LIST()()
     DECLARATION_STATEMENT(ARRAY(INT)[10])("a")()
     DECLARATION_STATEMENT(ARRAY(INT)[10])("b")()
@@ -40,6 +39,8 @@
         EXPRESSION()(ARRAY_INDEX)
          VARIABLE()("b")()
          VARIABLE()("i")()
+  ()("
+")()
   FUNCTION(ARRAY(INT)[10])("sort")()
    VARIABLE_LIST()()
     DECLARATION_STATEMENT(ARRAY(INT)[10])("a")()
