--- correctOutput/sorting.s5	2015-02-09 13:34:58.158174000 +0100
+++ testOutput/sorting.s5	2015-02-13 14:27:32.604498000 +0100
@@ -1,6 +1,8 @@
  PROGRAM()()
  FUNCTION_LIST()()
   FUNCTION(VOID)("main")()
+   TYPE(VOID)()
+   VARIABLE()("main")()
    (nil)
    STATEMENT_LIST()()
     DECLARATION_STATEMENT(ARRAY(INT)[10])("a")()
@@ -41,6 +43,8 @@
          VARIABLE()("b")()
          VARIABLE()("i")()
   FUNCTION(ARRAY(INT)[10])("sort")()
+   TYPE(ARRAY(INT)[10])()
+   VARIABLE()("sort")()
    VARIABLE_LIST()()
     DECLARATION_STATEMENT(ARRAY(INT)[10])("a")()
    STATEMENT_LIST()()
