--- correctOutput/strings_and_comments.s5	2015-02-09 13:34:58.168169000 +0100
+++ testOutput/strings_and_comments.s5	2015-02-13 14:35:16.653044000 +0100
@@ -2,16 +2,3 @@
  FUNCTION_LIST()()
   FUNCTION(INT)("main")()
    (nil)
-   STATEMENT_LIST()()
-    PRINT_STATEMENT()()
-     EXPRESSION_LIST()()
-      CONSTANT(STRING)("Hello, world!")()
-    PRINT_STATEMENT()()
-     EXPRESSION_LIST()()
-      CONSTANT(STRING)("")()
-    PRINT_STATEMENT()()
-     EXPRESSION_LIST()()
-      CONSTANT(STRING)("Hello, \"world\"!")()
-    PRINT_STATEMENT()()
-     EXPRESSION_LIST()()
-      CONSTANT(STRING)("//Comment")()
