--- correctOutput/type_errors_functions.s5	2015-02-09 13:34:58.198172000 +0100
+++ testOutput/type_errors_functions.s5	2015-02-13 13:27:37.438533000 +0100
@@ -1,7 +1,6 @@
  PROGRAM()()
  FUNCTION_LIST()()
   FUNCTION(VOID)("main")()
-   (nil)
    STATEMENT_LIST()()
     DECLARATION_STATEMENT(INT)("a")()
     DECLARATION_STATEMENT(BOOL)("b")()
@@ -33,12 +32,10 @@
       VARIABLE()("a")()
       VARIABLE()("b")()
   FUNCTION(VOID)("void_func")()
-   (nil)
    STATEMENT_LIST()()
     RETURN_STATEMENT()()
      CONSTANT(INT)(0)()
   FUNCTION(BOOL)("bool_func")()
-   (nil)
    STATEMENT_LIST()()
     RETURN_STATEMENT()()
      CONSTANT(BOOL)(False)()
