--- correctOutput/type_errors_arrays.s4	2015-02-09 13:34:58.175171000 +0100
+++ testOutput/type_errors_arrays.s4	2015-02-13 13:56:54.775280000 +0100
@@ -2,109 +2,109 @@
   Simplify FUNCTION 
    Simplify TYPE 
    Simplify STATEMENT_LIST 
-    Simplify STATEMENT_LIST 
      Simplify STATEMENT_LIST 
-      Simplify STATEMENT_LIST 
        Simplify STATEMENT_LIST 
-        Simplify STATEMENT_LIST 
          Simplify STATEMENT_LIST 
-          Simplify STATEMENT_LIST 
            Simplify STATEMENT_LIST 
-            Simplify STATEMENT_LIST 
              Simplify STATEMENT_LIST 
-              Simplify STATEMENT_LIST 
                Simplify STATEMENT_LIST 
-                Simplify STATEMENT_LIST 
+                 Simplify STATEMENT_LIST 
+                   Simplify STATEMENT_LIST 
+                     Simplify STATEMENT_LIST 
+                       Simplify STATEMENT_LIST 
+                         Simplify STATEMENT_LIST 
+                           Simplify STATEMENT_LIST 
+                             Simplify STATEMENT_LIST 
+                               Simplify STATEMENT 
+                                 Simplify DECLARATION_STATEMENT 
+                                   Simplify TYPE 
+                                     Simplify TYPE 
+                                     Simplify INDEX_LIST 
+                             Simplify STATEMENT 
+                               Simplify DECLARATION_STATEMENT 
+                                 Simplify TYPE 
+                                   Simplify TYPE 
+                                   Simplify INDEX_LIST 
+                                     Simplify INDEX_LIST 
+                           Simplify STATEMENT 
+                             Simplify DECLARATION_STATEMENT 
+                               Simplify TYPE 
+                                 Simplify TYPE 
+                                 Simplify INDEX_LIST 
+                         Simplify STATEMENT 
+                           Simplify DECLARATION_STATEMENT 
+                             Simplify TYPE 
+                               Simplify TYPE 
+                               Simplify INDEX_LIST 
+                                 Simplify INDEX_LIST 
+                                   Simplify INDEX_LIST 
+                       Simplify STATEMENT 
+                          Simplify EXPRESSION (VARIABLE) 
+                          Simplify EXPRESSION (NEW) 
+                            Simplify TYPE 
+                              Simplify TYPE 
+                              Simplify INDEX_LIST 
+                     Simplify STATEMENT 
+                        Simplify EXPRESSION (VARIABLE) 
+                        Simplify EXPRESSION (NEW) 
+                          Simplify TYPE 
+                            Simplify TYPE 
+                            Simplify INDEX_LIST 
+                   Simplify STATEMENT 
+                      Simplify EXPRESSION (VARIABLE) 
+                      Simplify EXPRESSION (NEW) 
+                        Simplify TYPE 
+                          Simplify TYPE 
+                          Simplify INDEX_LIST 
                  Simplify STATEMENT 
-                  Simplify DECLARATION_STATEMENT 
-                   Simplify TYPE 
+                    Simplify EXPRESSION (VARIABLE) 
+                    Simplify EXPRESSION (NEW) 
+                      Simplify TYPE 
+                        Simplify TYPE 
+                        Simplify INDEX_LIST 
+                          Simplify INDEX_LIST 
+               Simplify STATEMENT 
+                  Simplify EXPRESSION (VARIABLE) 
+                  Simplify EXPRESSION (NEW) 
                     Simplify TYPE 
-                    Simplify INDEX_LIST 
-                Simplify STATEMENT 
-                 Simplify DECLARATION_STATEMENT 
+                      Simplify TYPE 
+                      Simplify INDEX_LIST 
+                        Simplify INDEX_LIST 
+             Simplify STATEMENT 
+                Simplify EXPRESSION (VARIABLE) 
+                Simplify EXPRESSION (NEW) 
                   Simplify TYPE 
-                   Simplify TYPE 
-                   Simplify INDEX_LIST 
+                    Simplify TYPE 
                     Simplify INDEX_LIST 
-               Simplify STATEMENT 
-                Simplify DECLARATION_STATEMENT 
-                 Simplify TYPE 
-                  Simplify TYPE 
-                  Simplify INDEX_LIST 
-              Simplify STATEMENT 
-               Simplify DECLARATION_STATEMENT 
-                Simplify TYPE 
-                 Simplify TYPE 
-                 Simplify INDEX_LIST 
-                  Simplify INDEX_LIST 
-                   Simplify INDEX_LIST 
-             Simplify STATEMENT 
-               Simplify EXPRESSION (VARIABLE) 
-               Simplify EXPRESSION (NEW) 
-                Simplify TYPE 
-                 Simplify TYPE 
-                 Simplify INDEX_LIST 
-            Simplify STATEMENT 
+           Simplify STATEMENT 
               Simplify EXPRESSION (VARIABLE) 
               Simplify EXPRESSION (NEW) 
-               Simplify TYPE 
                 Simplify TYPE 
-                Simplify INDEX_LIST 
-           Simplify STATEMENT 
-             Simplify EXPRESSION (VARIABLE) 
-             Simplify EXPRESSION (NEW) 
-              Simplify TYPE 
-               Simplify TYPE 
-               Simplify INDEX_LIST 
-          Simplify STATEMENT 
+                  Simplify TYPE 
+                  Simplify INDEX_LIST 
+                    Simplify INDEX_LIST 
+         Simplify STATEMENT 
             Simplify EXPRESSION (VARIABLE) 
             Simplify EXPRESSION (NEW) 
-             Simplify TYPE 
               Simplify TYPE 
-              Simplify INDEX_LIST 
-               Simplify INDEX_LIST 
-         Simplify STATEMENT 
-           Simplify EXPRESSION (VARIABLE) 
-           Simplify EXPRESSION (NEW) 
-            Simplify TYPE 
-             Simplify TYPE 
-             Simplify INDEX_LIST 
-              Simplify INDEX_LIST 
-        Simplify STATEMENT 
+                Simplify TYPE 
+                Simplify INDEX_LIST 
+                  Simplify INDEX_LIST 
+                    Simplify INDEX_LIST 
+       Simplify STATEMENT 
           Simplify EXPRESSION (VARIABLE) 
           Simplify EXPRESSION (NEW) 
-           Simplify TYPE 
             Simplify TYPE 
-            Simplify INDEX_LIST 
-       Simplify STATEMENT 
-         Simplify EXPRESSION (VARIABLE) 
-         Simplify EXPRESSION (NEW) 
-          Simplify TYPE 
-           Simplify TYPE 
-           Simplify INDEX_LIST 
-            Simplify INDEX_LIST 
-      Simplify STATEMENT 
+              Simplify TYPE 
+              Simplify INDEX_LIST 
+                Simplify INDEX_LIST 
+                  Simplify INDEX_LIST 
+                    Simplify INDEX_LIST 
+     Simplify STATEMENT 
         Simplify EXPRESSION (VARIABLE) 
         Simplify EXPRESSION (NEW) 
-         Simplify TYPE 
           Simplify TYPE 
-          Simplify INDEX_LIST 
-           Simplify INDEX_LIST 
-            Simplify INDEX_LIST 
-     Simplify STATEMENT 
-       Simplify EXPRESSION (VARIABLE) 
-       Simplify EXPRESSION (NEW) 
-        Simplify TYPE 
-         Simplify TYPE 
-         Simplify INDEX_LIST 
-          Simplify INDEX_LIST 
-           Simplify INDEX_LIST 
+            Simplify TYPE 
             Simplify INDEX_LIST 
-    Simplify STATEMENT 
-      Simplify EXPRESSION (VARIABLE) 
-      Simplify EXPRESSION (NEW) 
-       Simplify TYPE 
-        Simplify TYPE 
-        Simplify INDEX_LIST 
-         Simplify INDEX_LIST 
-          Simplify INDEX_LIST 
+              Simplify INDEX_LIST 
+                Simplify INDEX_LIST 
