--- correctOutput/type_errors_arrays.s4	2015-02-09 13:34:58.175171000 +0100
+++ testOutput/type_errors_arrays.s4	2015-02-13 13:53:54.662277000 +0100
@@ -1,110 +1,110 @@
  Simplify FUNCTION_LIST 
-  Simplify FUNCTION 
-   Simplify TYPE 
-   Simplify STATEMENT_LIST 
+   Simplify FUNCTION 
+    Simplify TYPE 
     Simplify STATEMENT_LIST 
-     Simplify STATEMENT_LIST 
       Simplify STATEMENT_LIST 
-       Simplify STATEMENT_LIST 
         Simplify STATEMENT_LIST 
-         Simplify STATEMENT_LIST 
           Simplify STATEMENT_LIST 
-           Simplify STATEMENT_LIST 
             Simplify STATEMENT_LIST 
-             Simplify STATEMENT_LIST 
               Simplify STATEMENT_LIST 
-               Simplify STATEMENT_LIST 
                 Simplify STATEMENT_LIST 
-                 Simplify STATEMENT 
-                  Simplify DECLARATION_STATEMENT 
-                   Simplify TYPE 
-                    Simplify TYPE 
-                    Simplify INDEX_LIST 
+                  Simplify STATEMENT_LIST 
+                    Simplify STATEMENT_LIST 
+                      Simplify STATEMENT_LIST 
+                        Simplify STATEMENT_LIST 
+                          Simplify STATEMENT_LIST 
+                            Simplify STATEMENT_LIST 
+                              Simplify STATEMENT_LIST 
+                                Simplify STATEMENT 
+                                  Simplify DECLARATION_STATEMENT 
+                                    Simplify TYPE 
+                                      Simplify TYPE 
+                                      Simplify INDEX_LIST 
+                              Simplify STATEMENT 
+                                Simplify DECLARATION_STATEMENT 
+                                  Simplify TYPE 
+                                    Simplify TYPE 
+                                    Simplify INDEX_LIST 
+                                      Simplify INDEX_LIST 
+                            Simplify STATEMENT 
+                              Simplify DECLARATION_STATEMENT 
+                                Simplify TYPE 
+                                  Simplify TYPE 
+                                  Simplify INDEX_LIST 
+                          Simplify STATEMENT 
+                            Simplify DECLARATION_STATEMENT 
+                              Simplify TYPE 
+                                Simplify TYPE 
+                                Simplify INDEX_LIST 
+                                  Simplify INDEX_LIST 
+                                    Simplify INDEX_LIST 
+                        Simplify STATEMENT 
+                           Simplify EXPRESSION (VARIABLE) 
+                           Simplify EXPRESSION (NEW) 
+                             Simplify TYPE 
+                               Simplify TYPE 
+                               Simplify INDEX_LIST 
+                      Simplify STATEMENT 
+                         Simplify EXPRESSION (VARIABLE) 
+                         Simplify EXPRESSION (NEW) 
+                           Simplify TYPE 
+                             Simplify TYPE 
+                             Simplify INDEX_LIST 
+                    Simplify STATEMENT 
+                       Simplify EXPRESSION (VARIABLE) 
+                       Simplify EXPRESSION (NEW) 
+                         Simplify TYPE 
+                           Simplify TYPE 
+                           Simplify INDEX_LIST 
+                  Simplify STATEMENT 
+                     Simplify EXPRESSION (VARIABLE) 
+                     Simplify EXPRESSION (NEW) 
+                       Simplify TYPE 
+                         Simplify TYPE 
+                         Simplify INDEX_LIST 
+                           Simplify INDEX_LIST 
                 Simplify STATEMENT 
-                 Simplify DECLARATION_STATEMENT 
-                  Simplify TYPE 
-                   Simplify TYPE 
-                   Simplify INDEX_LIST 
-                    Simplify INDEX_LIST 
-               Simplify STATEMENT 
-                Simplify DECLARATION_STATEMENT 
-                 Simplify TYPE 
-                  Simplify TYPE 
-                  Simplify INDEX_LIST 
+                   Simplify EXPRESSION (VARIABLE) 
+                   Simplify EXPRESSION (NEW) 
+                     Simplify TYPE 
+                       Simplify TYPE 
+                       Simplify INDEX_LIST 
+                         Simplify INDEX_LIST 
               Simplify STATEMENT 
-               Simplify DECLARATION_STATEMENT 
-                Simplify TYPE 
-                 Simplify TYPE 
-                 Simplify INDEX_LIST 
-                  Simplify INDEX_LIST 
-                   Simplify INDEX_LIST 
-             Simplify STATEMENT 
+                 Simplify EXPRESSION (VARIABLE) 
+                 Simplify EXPRESSION (NEW) 
+                   Simplify TYPE 
+                     Simplify TYPE 
+                     Simplify INDEX_LIST 
+            Simplify STATEMENT 
                Simplify EXPRESSION (VARIABLE) 
                Simplify EXPRESSION (NEW) 
-                Simplify TYPE 
                  Simplify TYPE 
-                 Simplify INDEX_LIST 
-            Simplify STATEMENT 
-              Simplify EXPRESSION (VARIABLE) 
-              Simplify EXPRESSION (NEW) 
-               Simplify TYPE 
-                Simplify TYPE 
-                Simplify INDEX_LIST 
-           Simplify STATEMENT 
+                   Simplify TYPE 
+                   Simplify INDEX_LIST 
+                     Simplify INDEX_LIST 
+          Simplify STATEMENT 
              Simplify EXPRESSION (VARIABLE) 
              Simplify EXPRESSION (NEW) 
-              Simplify TYPE 
                Simplify TYPE 
-               Simplify INDEX_LIST 
-          Simplify STATEMENT 
-            Simplify EXPRESSION (VARIABLE) 
-            Simplify EXPRESSION (NEW) 
-             Simplify TYPE 
-              Simplify TYPE 
-              Simplify INDEX_LIST 
-               Simplify INDEX_LIST 
-         Simplify STATEMENT 
+                 Simplify TYPE 
+                 Simplify INDEX_LIST 
+                   Simplify INDEX_LIST 
+                     Simplify INDEX_LIST 
+        Simplify STATEMENT 
            Simplify EXPRESSION (VARIABLE) 
            Simplify EXPRESSION (NEW) 
-            Simplify TYPE 
              Simplify TYPE 
-             Simplify INDEX_LIST 
-              Simplify INDEX_LIST 
-        Simplify STATEMENT 
-          Simplify EXPRESSION (VARIABLE) 
-          Simplify EXPRESSION (NEW) 
-           Simplify TYPE 
-            Simplify TYPE 
-            Simplify INDEX_LIST 
-       Simplify STATEMENT 
+               Simplify TYPE 
+               Simplify INDEX_LIST 
+                 Simplify INDEX_LIST 
+                   Simplify INDEX_LIST 
+                     Simplify INDEX_LIST 
+      Simplify STATEMENT 
          Simplify EXPRESSION (VARIABLE) 
          Simplify EXPRESSION (NEW) 
-          Simplify TYPE 
            Simplify TYPE 
-           Simplify INDEX_LIST 
-            Simplify INDEX_LIST 
-      Simplify STATEMENT 
-        Simplify EXPRESSION (VARIABLE) 
-        Simplify EXPRESSION (NEW) 
-         Simplify TYPE 
-          Simplify TYPE 
-          Simplify INDEX_LIST 
-           Simplify INDEX_LIST 
-            Simplify INDEX_LIST 
-     Simplify STATEMENT 
-       Simplify EXPRESSION (VARIABLE) 
-       Simplify EXPRESSION (NEW) 
-        Simplify TYPE 
-         Simplify TYPE 
-         Simplify INDEX_LIST 
-          Simplify INDEX_LIST 
-           Simplify INDEX_LIST 
-            Simplify INDEX_LIST 
-    Simplify STATEMENT 
-      Simplify EXPRESSION (VARIABLE) 
-      Simplify EXPRESSION (NEW) 
-       Simplify TYPE 
-        Simplify TYPE 
-        Simplify INDEX_LIST 
-         Simplify INDEX_LIST 
-          Simplify INDEX_LIST 
+             Simplify TYPE 
+             Simplify INDEX_LIST 
+               Simplify INDEX_LIST 
+                 Simplify INDEX_LIST 
