--- correctOutput/variables.s4	2015-02-09 13:34:58.206169000 +0100
+++ testOutput/variables.s4	2015-02-13 13:27:37.557533000 +0100
@@ -2,26 +2,26 @@
   Simplify FUNCTION 
    Simplify TYPE 
    Simplify STATEMENT_LIST 
-    Simplify STATEMENT_LIST 
      Simplify STATEMENT_LIST 
-      Simplify STATEMENT_LIST 
        Simplify STATEMENT_LIST 
-        Simplify STATEMENT_LIST 
+         Simplify STATEMENT_LIST 
+           Simplify STATEMENT_LIST 
+             Simplify STATEMENT_LIST 
+               Simplify STATEMENT 
+                 Simplify DECLARATION_STATEMENT 
+                   Simplify TYPE 
+             Simplify STATEMENT 
+               Simplify DECLARATION_STATEMENT 
+                 Simplify TYPE 
+           Simplify STATEMENT 
+             Simplify DECLARATION_STATEMENT 
+               Simplify TYPE 
          Simplify STATEMENT 
-          Simplify DECLARATION_STATEMENT 
-           Simplify TYPE 
-        Simplify STATEMENT 
-         Simplify DECLARATION_STATEMENT 
-          Simplify TYPE 
+            Simplify EXPRESSION (VARIABLE) 
+            Simplify EXPRESSION (CONSTANT) 
        Simplify STATEMENT 
-        Simplify DECLARATION_STATEMENT 
-         Simplify TYPE 
-      Simplify STATEMENT 
+          Simplify EXPRESSION (VARIABLE) 
+          Simplify EXPRESSION (CONSTANT) 
+     Simplify STATEMENT 
         Simplify EXPRESSION (VARIABLE) 
         Simplify EXPRESSION (CONSTANT) 
-     Simplify STATEMENT 
-       Simplify EXPRESSION (VARIABLE) 
-       Simplify EXPRESSION (CONSTANT) 
-    Simplify STATEMENT 
-      Simplify EXPRESSION (VARIABLE) 
-      Simplify EXPRESSION (CONSTANT) 
