--- correctOutput/variables.s5	2015-02-09 13:34:58.207170000 +0100
+++ testOutput/variables.s5	2015-02-13 14:35:17.493045000 +0100
@@ -2,16 +2,3 @@
  FUNCTION_LIST()()
   FUNCTION(VOID)("main")()
    (nil)
-   STATEMENT_LIST()()
-    DECLARATION_STATEMENT(INT)("a")()
-    DECLARATION_STATEMENT(BOOL)("b")()
-    DECLARATION_STATEMENT(FLOAT)("c")()
-    ASSIGNMENT_STATEMENT()()
-     VARIABLE()("a")()
-     CONSTANT(INT)(10)()
-    ASSIGNMENT_STATEMENT()()
-     VARIABLE()("b")()
-     CONSTANT(BOOL)(True)()
-    ASSIGNMENT_STATEMENT()()
-     VARIABLE()("c")()
-     CONSTANT(FLOAT)(0.100000)()
