--- correctOutput/while_hard.s8	2015-02-09 13:34:58.218170000 +0100
+++ testOutput/while_hard.s8	2015-02-13 16:38:57.622220000 +0100
@@ -8,5 +8,23 @@
 .INTEGER: .ascii "%d \000"
 .FLOAT: .ascii "%f \000"
 .NEWLINE: .ascii "\n \000"
+.STRING0: .ascii (null)
+.ascii "\000"
+.STRING1: .ascii (null)
+.ascii "\000"
+.STRING2: .ascii (null)
+.ascii "\000"
+.STRING3: .ascii (null)
+.ascii "\000"
+.STRING4: .ascii (null)
+.ascii "\000"
+.STRING5: .ascii (null)
+.ascii "\000"
+.STRING6: .ascii (null)
+.ascii "\000"
+.STRING7: .ascii (null)
+.ascii "\000"
+.STRING8: .ascii (null)
+.ascii "\000"
 .globl main
 .align	2
