--- correctOutput/adding.s9	2015-02-22 13:09:53.007142000 +0100
+++ testOutput/adding.s9	2015-02-22 13:42:02.091346000 +0100
@@ -1,14 +0,0 @@
- PROGRAM
- FUNCTION_LIST
-  FUNCTION, ("adding"), (nil)
-   (nil)
-   STATEMENT_LIST
-    DECLARATION_STATEMENT
-    ASSIGNMENT_STATEMENT
-     VARIABLE, ("a"), depth: 2, stack_offset: -4, entry label: "a"
-     EXPRESSION
-      CONSTANT
-      CONSTANT
-    PRINT_STATEMENT
-     EXPRESSION_LIST
-      VARIABLE, ("a"), depth: 2, stack_offset: -4, entry label: "a"
