--- correctOutput/arithmetic_float.s10	2015-02-22 13:09:53.011143000 +0100
+++ testOutput/arithmetic_float.s10	2015-02-22 15:22:00.476142000 +0100
@@ -1,73 +1,31 @@
 Type checking assignment
 Type checking assignment
 Type checking assignment
-Type checking expression UMINUS
 Type checking assignment
-Type checking expression UMINUS
 Type checking assignment
-Type checking expression UMINUS
 Type checking assignment
-Type checking expression UMINUS
 Type checking assignment
-Type checking expression +
 Type checking assignment
-Type checking expression +
 Type checking assignment
-Type checking expression +
 Type checking assignment
-Type checking expression -
 Type checking assignment
-Type checking expression -
 Type checking assignment
-Type checking expression -
 Type checking assignment
-Type checking expression -
-Type checking expression UMINUS
 Type checking assignment
-Type checking expression -
-Type checking expression UMINUS
 Type checking assignment
-Type checking expression -
-Type checking expression UMINUS
 Type checking assignment
-Type checking expression *
 Type checking assignment
-Type checking expression *
 Type checking assignment
-Type checking expression *
 Type checking assignment
-Type checking expression /
 Type checking assignment
-Type checking expression /
 Type checking assignment
-Type checking expression /
 Type checking assignment
-Type checking expression +
-Type checking expression +
 Type checking assignment
-Type checking expression -
-Type checking expression -
 Type checking assignment
-Type checking expression *
-Type checking expression *
 Type checking assignment
-Type checking expression /
-Type checking expression /
 Type checking assignment
-Type checking expression +
-Type checking expression *
 Type checking assignment
-Type checking expression +
-Type checking expression *
 Type checking assignment
-Type checking expression +
-Type checking expression /
 Type checking assignment
-Type checking expression +
-Type checking expression /
 Type checking assignment
-Type checking expression -
-Type checking expression /
 Type checking assignment
-Type checking expression -
-Type checking expression /
