--- correctOutput/arithmetic_int.s10	2015-02-22 13:09:53.048142000 +0100
+++ testOutput/arithmetic_int.s10	2015-02-22 23:28:42.964301000 +0100
@@ -2,6 +2,10 @@
 Type checking assignment
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression UMINUS
 Type checking assignment
