--- correctOutput/arithmetic_float.s10	2015-02-22 13:09:53.011143000 +0100
+++ testOutput/arithmetic_float.s10	2015-02-22 23:09:13.684250000 +0100
@@ -2,6 +2,10 @@
 Type checking assignment
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression UMINUS
 Type checking assignment
@@ -10,64 +14,134 @@
 Type checking expression UMINUS
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression -
+Type error at:
+EXPRESSION (-)
 Type checking assignment
 Type checking expression -
+Type error at:
+EXPRESSION (-)
 Type checking assignment
 Type checking expression -
+Type error at:
+EXPRESSION (-)
 Type checking assignment
 Type checking expression -
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (-)
 Type checking assignment
 Type checking expression -
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (-)
 Type checking assignment
 Type checking expression -
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (-)
 Type checking assignment
 Type checking expression *
+Type error at:
+EXPRESSION (*)
 Type checking assignment
 Type checking expression *
+Type error at:
+EXPRESSION (*)
 Type checking assignment
 Type checking expression *
+Type error at:
+EXPRESSION (*)
 Type checking assignment
 Type checking expression /
+Type error at:
+EXPRESSION (/)
 Type checking assignment
 Type checking expression /
+Type error at:
+EXPRESSION (/)
 Type checking assignment
 Type checking expression /
+Type error at:
+EXPRESSION (/)
 Type checking assignment
 Type checking expression +
 Type checking expression +
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression -
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (-)
 Type checking assignment
 Type checking expression *
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
 Type checking assignment
 Type checking expression /
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
 Type checking assignment
 Type checking expression +
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression +
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression +
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression +
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
 Type checking assignment
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
