--- correctOutput/array_hard.s7	2015-02-22 13:09:53.120145000 +0100
+++ testOutput/array_hard.s7	2015-02-22 15:04:04.150468000 +0100
@@ -14,14 +14,3 @@
 Inserting variable c ARRAY(INT)[3] depth:2 offset:-12
 Retrieving variable c
 Retrieving function create
-Retrieving variable c
-Remove scope, index: 1 
-Add scope, index: 2 
-Inserting variable a ARRAY(INT)[3] depth:2 offset:-4
-Retrieving variable a
-Retrieving variable a
-Retrieving variable a
-Retrieving variable a
-Retrieving variable a
-Remove scope, index: 1 
-Remove scope, index: 0 
