--- correctOutput/array_hard.s10	2015-02-22 13:09:53.106144000 +0100
+++ testOutput/array_hard.s10	2015-02-24 13:46:06.710487000 +0100
@@ -5,22 +5,12 @@
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
 Type checking assignment
 Type checking expression NEW
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
