--- correctOutput/array_forloop.s10	2015-02-22 13:09:53.075144000 +0100
+++ testOutput/array_forloop.s10	2015-02-22 23:05:15.099017000 +0100
@@ -1,14 +1,27 @@
 Type checking assignment
 Type checking expression NEW
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression +
 Type checking expression +
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking assignment
@@ -17,9 +30,19 @@
 Type checking expression *
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (*)
 Type checking assignment
 Type checking expression +
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (+)
