--- correctOutput/array_types.s10	2015-02-22 13:09:53.172145000 +0100
+++ testOutput/array_types.s10	2015-02-23 15:32:25.080630000 +0100
@@ -1,7 +1,9 @@
 Type checking assignment
 Type checking expression NEW
+Type checking expression NEW
 Type checking assignment
 Type checking expression NEW
+Type checking expression NEW
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
@@ -9,9 +11,9 @@
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type checking expression ARRAY_INDEX
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
