--- correctOutput/array_types.s10	2015-02-22 13:09:53.172145000 +0100
+++ testOutput/array_types.s10	2015-02-22 23:53:49.569122000 +0100
@@ -6,19 +6,41 @@
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
