--- correctOutput/array_forloop.s10	2015-02-22 13:09:53.075144000 +0100
+++ testOutput/array_forloop.s10	2015-02-23 15:32:23.589616000 +0100
@@ -1,10 +1,15 @@
 Type checking assignment
 Type checking expression NEW
+Type checking expression NEW
 Type checking assignment
 Type checking assignment
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type checking expression +
+Type checking expression +
+Type checking expression *
+Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking expression +
 Type checking expression +
@@ -17,9 +22,13 @@
 Type checking expression *
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type checking expression *
+Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking assignment
 Type checking expression +
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type checking expression +
+Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
