--- correctOutput/compare_float.s10	2015-02-22 13:09:53.195144000 +0100
+++ testOutput/compare_float.s10	2015-02-24 13:32:23.120117000 +0100
@@ -4,38 +4,74 @@
 Type checking assignment
 Type checking assignment
 Type checking expression >
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression >
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression >
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking expression >=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression >=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression >=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ==
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ==
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ==
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression !=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression !=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression !=
+Type error at:
+ASSIGNMENT_STATEMENT
