--- correctOutput/compare_float.s10	2015-02-22 13:09:53.195144000 +0100
+++ testOutput/compare_float.s10	2015-02-22 15:53:49.999295000 +0100
@@ -1,41 +1,22 @@
 Type checking assignment
 Type checking assignment
-Type checking expression UMINUS
 Type checking assignment
 Type checking assignment
-Type checking expression >
 Type checking assignment
-Type checking expression >
 Type checking assignment
-Type checking expression >
 Type checking assignment
-Type checking expression <
 Type checking assignment
-Type checking expression <
 Type checking assignment
-Type checking expression <
 Type checking assignment
 Type checking assignment
-Type checking expression >=
 Type checking assignment
-Type checking expression >=
 Type checking assignment
-Type checking expression >=
 Type checking assignment
-Type checking expression <=
 Type checking assignment
-Type checking expression <=
 Type checking assignment
-Type checking expression <=
 Type checking assignment
-Type checking expression ==
 Type checking assignment
-Type checking expression ==
 Type checking assignment
-Type checking expression ==
 Type checking assignment
-Type checking expression !=
 Type checking assignment
-Type checking expression !=
 Type checking assignment
-Type checking expression !=
