--- correctOutput/compare_int.s10	2015-02-22 13:09:53.231147000 +0100
+++ testOutput/compare_int.s10	2015-02-24 17:43:09.747972000 +0100
@@ -4,38 +4,74 @@
 Type checking assignment
 Type checking assignment
 Type checking expression >
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression >
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression >
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking expression >=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression >=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression >=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression <=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ==
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ==
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ==
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression !=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression !=
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression !=
+Type error at:
+ASSIGNMENT_STATEMENT
