--- correctOutput/compare_float.s10	2015-02-22 13:09:53.195144000 +0100
+++ testOutput/compare_float.s10	2015-02-22 23:09:15.010256000 +0100
@@ -1,6 +1,10 @@
 Type checking assignment
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking expression >
@@ -29,13 +33,25 @@
 Type checking expression <=
 Type checking assignment
 Type checking expression ==
+Type error at:
+EXPRESSION (==)
 Type checking assignment
 Type checking expression ==
+Type error at:
+EXPRESSION (==)
 Type checking assignment
 Type checking expression ==
+Type error at:
+EXPRESSION (==)
 Type checking assignment
 Type checking expression !=
+Type error at:
+EXPRESSION (!=)
 Type checking assignment
 Type checking expression !=
+Type error at:
+EXPRESSION (!=)
 Type checking assignment
 Type checking expression !=
+Type error at:
+EXPRESSION (!=)
