--- correctOutput/euclid.s10	2015-02-24 17:24:07.034923000 +0100
+++ testOutput/euclid.s10	2015-02-24 18:55:39.608029000 +0100
@@ -1,9 +1,12 @@
 Type checking expression -
 Type checking expression FUNC_CALL
+INT
 Type checking expression FUNC_CALL
+INT
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking expression -
 Type checking expression *
 Type checking expression /
+INT
 Type checking assignment
