--- correctOutput/euclid.s7	2015-02-22 13:09:53.271144000 +0100
+++ testOutput/euclid.s7	2015-02-22 15:09:22.804107000 +0100
@@ -6,8 +6,6 @@
 Inserting variable a INT depth:2 offset:12
 Inserting variable b INT depth:2 offset:8
 Retrieving function gcd
-Retrieving variable a
-Retrieving variable b
 Add strings ("Greatest common divisor of"), index: 0 
 Retrieving variable a
 Add strings ("and"), index: 1 
@@ -15,8 +13,6 @@
 Add strings ("is"), index: 2 
 Retrieving function gcd
 Retrieving variable a
-Retrieving variable b
-Retrieving variable a
 Add strings ("and"), index: 3 
 Retrieving variable b
 Add strings ("are relative primes"), index: 4 
@@ -28,11 +24,6 @@
 Retrieving variable b
 Retrieving variable g
 Retrieving function gcd
-Retrieving variable b
-Retrieving variable a
-Retrieving variable a
-Retrieving variable b
-Retrieving variable b
 Retrieving variable g
 Retrieving variable a
 Retrieving variable g
