--- correctOutput/euclid.s10	2015-02-22 13:09:53.257144000 +0100
+++ testOutput/euclid.s10	2015-02-22 15:53:50.318296000 +0100
@@ -1,12 +1,4 @@
 Type checking expression -
 Type checking expression FUNC_CALL
-Type checking expression FUNC_CALL
 Type checking assignment
-Type checking expression FUNC_CALL
-Type checking expression -
-Type checking expression *
-Type checking expression /
-Type checking expression -
-Type checking expression *
-Type checking expression /
 Type checking assignment
