--- correctOutput/euclid.s10	2015-02-22 13:09:53.257144000 +0100
+++ testOutput/euclid.s10	2015-02-23 15:16:16.199265000 +0100
@@ -1,12 +1,22 @@
 Type checking expression -
 Type checking expression FUNC_CALL
 Type checking expression FUNC_CALL
+Type checking expression FUNC_CALL
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking expression -
 Type checking expression *
 Type checking expression /
+Type checking expression /
+Type checking expression *
+Type checking expression /
+Type checking expression /
+Type checking expression FUNC_CALL
 Type checking expression -
 Type checking expression *
 Type checking expression /
+Type checking expression /
+Type checking expression *
+Type checking expression /
+Type checking expression /
 Type checking assignment
