--- correctOutput/euclid.s10	2015-02-22 13:09:53.257144000 +0100
+++ testOutput/euclid.s10	2015-02-23 15:19:20.865276000 +0100
@@ -1,12 +1,18 @@
 Type checking expression -
 Type checking expression FUNC_CALL
+Type error at:
+EXPRESSION (FUNC_CALL)
+Type error at:
+EXPRESSION (FUNC_CALL)
 Type checking expression FUNC_CALL
+Type error at:
+EXPRESSION (FUNC_CALL)
+Type error at:
+EXPRESSION (FUNC_CALL)
 Type checking assignment
 Type checking expression FUNC_CALL
-Type checking expression -
-Type checking expression *
-Type checking expression /
-Type checking expression -
-Type checking expression *
-Type checking expression /
+Type error at:
+EXPRESSION (FUNC_CALL)
+Type error at:
+EXPRESSION (FUNC_CALL)
 Type checking assignment
