--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-22 17:19:20.000835000 +0100
@@ -1,81 +1,215 @@
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (==)
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!=)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!=)
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (==)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (==)
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!=)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
 Type checking expression +
 Type checking expression /
 Type checking expression /
-Type checking expression UMINUS
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (+)
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (-)
 Type checking expression *
 Type checking expression *
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression *
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
 Type checking expression UMINUS
-Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (-)
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (+)
 Type checking expression +
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (*)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression /
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
 Type checking expression /
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
 Type checking expression &&
+Type error at:
+EXPRESSION (&&)
 Type checking expression &&
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
 Type checking expression &&
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
 Type checking expression &&
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -83,61 +217,179 @@
 Type checking expression !
 Type checking expression !
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
 Type checking expression !
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (&&)
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
 Type checking expression ||
 Type checking expression &&
+Type error at:
+EXPRESSION (&&)
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<=)
 Type checking expression &&
 Type checking expression !
 Type checking expression &&
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (!)
 Type checking expression ||
 Type checking expression !
 Type checking expression !=
-Type checking expression !
+Type error at:
+EXPRESSION (!=)
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression *
 Type checking expression *
 Type checking expression +
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression /
 Type checking expression /
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
+Type error at:
+EXPRESSION (||)
 Type checking expression &&
 Type checking expression &&
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression ||
 Type checking expression ||
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking expression &&
 Type checking expression >=
+Type error at:
+EXPRESSION (>=)
 Type checking expression !=
+Type error at:
+EXPRESSION (!=)
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<=)
 Type checking expression !
 Type checking expression <
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<)
+Type error at:
+ASSIGNMENT_STATEMENT
