--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-23 15:13:18.813274000 +0100
@@ -9,16 +9,31 @@
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression *
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression *
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression *
 Type checking assignment
 Type checking expression ||
@@ -26,12 +41,30 @@
 Type checking expression UMINUS
 Type checking expression !=
 Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression ==
 Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
@@ -40,16 +73,67 @@
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
 Type checking expression +
 Type checking expression /
 Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression UMINUS
+Type checking expression +
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression -
+Type checking expression +
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression UMINUS
+Type checking expression +
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
 Type checking expression UMINUS
 Type checking expression *
 Type checking expression *
 Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
 Type checking assignment
 Type checking expression *
 Type checking expression +
@@ -57,17 +141,87 @@
 Type checking expression UMINUS
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression +
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression +
 Type checking assignment
 Type checking expression /
 Type checking expression -
 Type checking expression /
 Type checking expression /
+Type checking expression /
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression /
 Type checking expression /
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -76,6 +230,28 @@
 Type checking expression &&
 Type checking expression &&
 Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -85,49 +261,275 @@
 Type checking expression !
 Type checking expression !
 Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression ||
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
 Type checking expression &&
 Type checking expression !
+Type checking expression !
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression >
 Type checking expression <=
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression &&
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type checking expression !
+Type checking expression ||
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression &&
 Type checking expression !
 Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
 Type checking expression ||
 Type checking expression !
 Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
 Type checking expression !
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression *
+Type checking expression *
+Type checking expression *
 Type checking assignment
 Type checking expression *
 Type checking expression *
 Type checking expression +
+Type checking expression +
+Type checking expression *
+Type checking expression +
+Type checking expression +
 Type checking assignment
 Type checking expression /
 Type checking expression /
 Type checking expression -
+Type checking expression -
+Type checking expression /
+Type checking expression -
+Type checking expression -
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression ||
 Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -136,8 +538,128 @@
 Type checking expression &&
 Type checking expression >=
 Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression !
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (!)
+Type checking expression ||
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
 Type checking expression <=
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
 Type checking expression !
 Type checking expression <
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
