--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-24 17:36:05.597728000 +0100
@@ -26,12 +26,22 @@
 Type checking expression UMINUS
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+, Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+, Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
@@ -40,6 +50,11 @@
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+, Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
@@ -97,6 +112,9 @@
 Type checking expression >
 Type checking expression <=
 Type checking expression UMINUS
+Not bool
+Type error at:
+EXPRESSION (||)
 Type checking expression &&
 Type checking expression !
 Type checking expression &&
@@ -104,6 +122,11 @@
 Type checking expression !
 Type checking expression !=
 Type checking expression !
+Type error at:
+EXPRESSION (||)
+Unequal types
+, Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
