--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-22 17:07:49.701125000 +0100
@@ -1,73 +1,195 @@
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
-Type checking expression *
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (*)
+Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
-Type checking expression *
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (*)
+Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
-Type checking expression *
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (*)
+Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
 Type checking expression +
 Type checking expression /
 Type checking expression /
-Type checking expression UMINUS
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (+)
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (-)
 Type checking expression *
 Type checking expression *
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression *
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
 Type checking expression UMINUS
-Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (-)
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (+)
 Type checking expression +
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (*)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression /
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
 Type checking expression /
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -76,6 +198,8 @@
 Type checking expression &&
 Type checking expression &&
 Type checking expression &&
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -83,51 +207,105 @@
 Type checking expression !
 Type checking expression !
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
 Type checking expression !
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression >
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
 Type checking expression &&
 Type checking expression !
 Type checking expression &&
+Type error at:
+EXPRESSION (!)
 Type checking expression ||
 Type checking expression !
 Type checking expression !=
+Type error at:
+EXPRESSION (!)
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression *
 Type checking expression *
 Type checking expression +
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression /
 Type checking expression /
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression ||
 Type checking expression ||
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -138,6 +316,14 @@
 Type checking expression !=
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
 Type checking expression !
 Type checking expression <
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
