--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-22 23:56:20.806908000 +0100
@@ -26,12 +26,20 @@
 Type checking expression UMINUS
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
@@ -40,6 +48,10 @@
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
@@ -103,7 +115,13 @@
 Type checking expression ||
 Type checking expression !
 Type checking expression !=
+Type error at:
+EXPRESSION (!)
 Type checking expression !
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (&&)
 Type checking assignment
 Type checking expression +
 Type checking expression *
