--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-24 13:35:41.387168000 +0100
@@ -26,12 +26,22 @@
 Type checking expression UMINUS
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
@@ -40,6 +50,11 @@
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
@@ -97,13 +112,29 @@
 Type checking expression >
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
 Type checking expression &&
 Type checking expression !
 Type checking expression &&
 Type checking expression ||
 Type checking expression !
 Type checking expression !=
+Type error at:
+EXPRESSION (!)
 Type checking expression !
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
@@ -136,8 +167,24 @@
 Type checking expression &&
 Type checking expression >=
 Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
 Type checking expression !
 Type checking expression <
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
