--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-22 16:54:42.056889000 +0100
@@ -1,37 +1,61 @@
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
 Type checking expression *
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
 Type checking expression *
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
 Type checking expression *
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
@@ -40,6 +64,8 @@
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
@@ -50,6 +76,8 @@
 Type checking expression *
 Type checking expression *
 Type checking expression *
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression *
 Type checking expression +
@@ -58,6 +86,8 @@
 Type checking expression UMINUS
 Type checking expression UMINUS
 Type checking expression +
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression /
 Type checking expression -
@@ -68,6 +98,8 @@
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -76,6 +108,8 @@
 Type checking expression &&
 Type checking expression &&
 Type checking expression &&
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -83,51 +117,87 @@
 Type checking expression !
 Type checking expression !
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
 Type checking expression !
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression >
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
 Type checking expression &&
 Type checking expression !
 Type checking expression &&
+Type error at:
+EXPRESSION (!)
 Type checking expression ||
 Type checking expression !
 Type checking expression !=
+Type error at:
+EXPRESSION (!)
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression *
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression *
 Type checking expression *
 Type checking expression +
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression /
 Type checking expression /
 Type checking expression -
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression ||
 Type checking expression ||
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -138,6 +208,14 @@
 Type checking expression !=
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
 Type checking expression !
 Type checking expression <
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
