--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-23 15:16:16.701252000 +0100
@@ -1,6 +1,7 @@
 Type checking assignment
 Type checking assignment
 Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking assignment
 Type checking assignment
 Type checking assignment
@@ -9,135 +10,1313 @@
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression +
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression *
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression +
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression *
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression +
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression *
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression ||
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression !=
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression ||
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression ==
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression ||
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression !=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ==
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
 Type checking expression +
 Type checking expression /
 Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression UMINUS
+Type checking expression +
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
 Type checking expression UMINUS
 Type checking expression *
 Type checking expression *
 Type checking expression *
-Type checking assignment
 Type checking expression *
-Type checking expression +
+Type checking expression *
+Type checking expression *
+Type checking expression *
 Type checking expression -
-Type checking expression UMINUS
-Type checking expression UMINUS
+Type checking expression +
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
 Type checking expression UMINUS
 Type checking expression +
-Type checking assignment
 Type checking expression /
-Type checking expression -
 Type checking expression /
 Type checking expression /
-Type checking expression -
 Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
 Type checking expression +
 Type checking expression -
+Type checking expression +
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
 Type checking expression UMINUS
-Type checking assignment
-Type checking expression ||
-Type checking expression ||
-Type checking expression ||
-Type checking expression &&
-Type checking expression &&
-Type checking expression &&
-Type checking expression &&
-Type checking assignment
-Type checking expression ||
-Type checking expression ||
-Type checking expression &&
-Type checking expression !
-Type checking expression !
-Type checking expression !
-Type checking expression !
-Type checking expression !
-Type checking expression &&
-Type checking expression !
-Type checking expression ||
-Type checking expression &&
-Type checking expression &&
-Type checking expression !
-Type checking assignment
-Type checking expression ||
-Type checking expression ||
-Type checking expression >
-Type checking expression <=
-Type checking expression UMINUS
-Type checking expression &&
-Type checking expression !
-Type checking expression &&
-Type checking expression ||
-Type checking expression !
-Type checking expression !=
-Type checking expression !
-Type checking assignment
 Type checking expression +
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression UMINUS
+Type checking expression *
 Type checking expression *
 Type checking expression *
-Type checking assignment
 Type checking expression *
 Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression -
 Type checking expression +
-Type checking assignment
 Type checking expression /
 Type checking expression /
-Type checking expression -
-Type checking assignment
-Type checking expression ||
-Type checking expression ||
-Type checking expression ||
-Type checking expression &&
-Type checking expression &&
-Type checking assignment
-Type checking expression ||
-Type checking expression &&
-Type checking expression &&
-Type checking expression ||
-Type checking expression ||
-Type checking assignment
-Type checking expression ||
-Type checking expression ||
-Type checking expression ||
-Type checking expression >
-Type checking expression &&
-Type checking expression >=
-Type checking expression !=
-Type checking expression <=
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression UMINUS
+Type checking expression +
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression UMINUS
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking assignment
+Type checking expression *
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression +
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression +
+Type checking expression *
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression +
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression +
+Type checking assignment
+Type checking expression /
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression /
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression /
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression +
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression -
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking assignment
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking assignment
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression ||
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression ||
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression !
+Type checking expression !
+Type checking assignment
+Type checking expression ||
+Type checking expression ||
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression &&
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type checking expression !
+Type checking expression ||
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression &&
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type checking expression !
+Type checking expression ||
+Type checking expression ||
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression &&
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type checking expression !
+Type checking expression ||
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression >
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression &&
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type checking expression !
+Type checking expression !
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type error at:
+EXPRESSION (!)
+Type checking expression !
+Type checking expression !
+Type checking expression !=
+Type checking expression !=
+Type checking expression !
+Type checking assignment
+Type checking expression +
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression +
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking expression *
+Type checking assignment
+Type checking expression *
+Type checking expression *
+Type checking expression +
+Type checking expression +
+Type checking expression *
+Type checking expression +
+Type checking expression +
+Type checking expression *
+Type checking expression *
+Type checking expression +
+Type checking expression +
+Type checking expression *
+Type checking expression +
+Type checking expression +
+Type checking assignment
+Type checking expression /
+Type checking expression /
+Type checking expression -
+Type checking expression -
+Type checking expression /
+Type checking expression -
+Type checking expression -
+Type checking expression /
+Type checking expression /
+Type checking expression -
+Type checking expression -
+Type checking expression /
+Type checking expression -
+Type checking expression -
+Type checking assignment
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression &&
+Type checking assignment
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression &&
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking assignment
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression !
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (!)
+Type checking expression ||
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression !
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression ||
+Type checking expression ||
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression !
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (!)
+Type checking expression ||
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression ||
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type checking expression >
+Type checking expression &&
+Type checking expression >=
+Type checking expression !=
+Type checking expression >=
+Type checking expression !=
+Type error at:
+EXPRESSION (&&)
+Unequal types
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression <=
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type checking expression !
+Type checking expression <
+Type checking expression UMINUS
+Type checking expression UMINUS
+Type checking expression <
 Type checking expression UMINUS
-Type checking expression !
-Type checking expression <
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (||)
+Unequal types
+Type error at:
+ASSIGNMENT_STATEMENT
