--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-22 23:10:44.402718000 +0100
@@ -1,6 +1,10 @@
 Type checking assignment
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking assignment
@@ -24,22 +28,46 @@
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (==)
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!=)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!=)
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (==)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (==)
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!=)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
@@ -97,12 +125,18 @@
 Type checking expression >
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<=)
 Type checking expression &&
 Type checking expression !
 Type checking expression &&
 Type checking expression ||
 Type checking expression !
 Type checking expression !=
+Type error at:
+EXPRESSION (!=)
 Type checking expression !
 Type checking assignment
 Type checking expression +
@@ -120,14 +154,34 @@
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
+Type error at:
+EXPRESSION (||)
 Type checking expression &&
 Type checking expression &&
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression ||
 Type checking expression ||
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -135,9 +189,21 @@
 Type checking expression >
 Type checking expression &&
 Type checking expression >=
+Type error at:
+EXPRESSION (>=)
 Type checking expression !=
+Type error at:
+EXPRESSION (!=)
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<=)
 Type checking expression !
 Type checking expression <
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<)
