--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-22 17:12:17.110556000 +0100
@@ -1,37 +1,79 @@
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
@@ -40,34 +82,92 @@
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
 Type checking expression +
 Type checking expression /
 Type checking expression /
-Type checking expression UMINUS
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (+)
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (-)
 Type checking expression *
 Type checking expression *
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression *
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
 Type checking expression UMINUS
-Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (-)
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (+)
 Type checking expression +
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (*)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression /
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
 Type checking expression /
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -76,6 +176,8 @@
 Type checking expression &&
 Type checking expression &&
 Type checking expression &&
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -83,61 +185,133 @@
 Type checking expression !
 Type checking expression !
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
 Type checking expression !
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+EXPRESSION (!)
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<=)
 Type checking expression &&
 Type checking expression !
 Type checking expression &&
+Type error at:
+EXPRESSION (!)
 Type checking expression ||
 Type checking expression !
 Type checking expression !=
+Type error at:
+EXPRESSION (!)
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression *
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression *
 Type checking expression *
 Type checking expression +
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression /
 Type checking expression /
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression ||
 Type checking expression ||
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking expression &&
 Type checking expression >=
+Type error at:
+EXPRESSION (>=)
 Type checking expression !=
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<=)
 Type checking expression !
 Type checking expression <
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<)
+Type error at:
+ASSIGNMENT_STATEMENT
