--- correctOutput/expressions.s10	2015-02-22 13:09:53.291144000 +0100
+++ testOutput/expressions.s10	2015-02-22 22:38:32.850756000 +0100
@@ -1,6 +1,10 @@
 Type checking assignment
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking assignment
@@ -9,65 +13,147 @@
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression +
 Type checking expression *
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (==)
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!=)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!=)
 Type checking expression ==
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (==)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ==
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (==)
 Type checking expression !=
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (!=)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
 Type checking expression -
 Type checking expression +
 Type checking expression /
 Type checking expression /
-Type checking expression UMINUS
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (+)
+Type checking expression UMINUS
+Type error at:
+EXPRESSION (-)
 Type checking expression *
 Type checking expression *
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression *
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (-)
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (+)
 Type checking expression +
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (*)
 Type checking assignment
 Type checking expression /
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
 Type checking expression /
 Type checking expression -
 Type checking expression /
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (-)
 Type checking expression +
 Type checking expression -
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
@@ -95,49 +181,109 @@
 Type checking expression ||
 Type checking expression ||
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<=)
 Type checking expression &&
 Type checking expression !
 Type checking expression &&
 Type checking expression ||
 Type checking expression !
 Type checking expression !=
+Type error at:
+EXPRESSION (!=)
 Type checking expression !
 Type checking assignment
 Type checking expression +
 Type checking expression *
+Type error at:
+EXPRESSION (*)
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression *
 Type checking expression *
 Type checking expression +
+Type error at:
+EXPRESSION (+)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
 Type checking assignment
 Type checking expression /
 Type checking expression /
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (/)
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
+Type error at:
+EXPRESSION (||)
 Type checking expression &&
 Type checking expression &&
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
 Type checking expression &&
 Type checking expression ||
 Type checking expression ||
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (||)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (&&)
+Type error at:
+EXPRESSION (||)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
 Type checking expression ||
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking expression &&
 Type checking expression >=
+Type error at:
+EXPRESSION (>=)
 Type checking expression !=
+Type error at:
+EXPRESSION (!=)
 Type checking expression <=
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<=)
 Type checking expression !
 Type checking expression <
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+EXPRESSION (<)
