--- correctOutput/fibonacci_recursive.s6	2015-02-22 13:09:53.355145000 +0100
+++ testOutput/fibonacci_recursive.s6	2015-02-22 15:04:06.234439000 +0100
@@ -4,42 +4,4 @@
 DECLARATION: parameter/variable : 'f', offset: -4
 VARIABLE: access: f
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: n
-EXPRESSION: End
-CONSTANT
-VARIABLE: access: n
-CONSTANT
-VARIABLE: access: f
-CONSTANT
-FUNCTION: End
-FUNCTION: Start: fibonacci_number
-DECLARATION: parameter/variable : 'n', offset: 8
-DECLARATION: parameter/variable : 'y', offset: -4
-VARIABLE: access: y
-CONSTANT
-EXPRESSION: Start: -
-VARIABLE: access: n
-CONSTANT
-EXPRESSION: End
-EXPRESSION: Start: -
-VARIABLE: access: n
-CONSTANT
-EXPRESSION: End
-VARIABLE: access: y
-EXPRESSION: Start: +
-EXPRESSION: Start: FUNC_CALL
-EXPRESSION: Start: -
-VARIABLE: access: n
-CONSTANT
-EXPRESSION: End
-EXPRESSION: End
-EXPRESSION: Start: FUNC_CALL
-EXPRESSION: Start: -
-VARIABLE: access: n
-CONSTANT
-EXPRESSION: End
-EXPRESSION: End
-EXPRESSION: End
-VARIABLE: access: y
-FUNCTION: End
-FUNCTION_LIST: End
+VARIABLE: access: fibonacci_number
