--- correctOutput/fibonacci_recursive.s7	2015-02-22 13:09:53.357147000 +0100
+++ testOutput/fibonacci_recursive.s7	2015-02-22 15:09:23.725130000 +0100
@@ -7,7 +7,6 @@
 Inserting variable f INT depth:2 offset:-4
 Retrieving variable f
 Retrieving function fibonacci_number
-Retrieving variable n
 Add strings ("The"), index: 0 
 Retrieving variable n
 Add strings ("th fibonacci number is"), index: 1 
@@ -21,9 +20,7 @@
 Retrieving variable n
 Retrieving variable y
 Retrieving function fibonacci_number
-Retrieving variable n
 Retrieving function fibonacci_number
-Retrieving variable n
 Retrieving variable y
 Remove scope, index: 1 
 Remove scope, index: 0 
