--- correctOutput/fibonacci_iterative.s10	2015-02-22 13:09:53.321144000 +0100
+++ testOutput/fibonacci_iterative.s10	2015-02-22 23:05:16.839029000 +0100
@@ -2,11 +2,19 @@
 Type checking assignment
 Type checking assignment
 Type checking expression >=
+Type error at:
+EXPRESSION (>=)
 Type checking assignment
 Type checking expression <
+Type error at:
+EXPRESSION (<)
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking assignment
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
