--- correctOutput/fibonacci_recursive.s9	2015-02-22 13:09:53.361145000 +0100
+++ testOutput/fibonacci_recursive.s9	2015-02-22 15:09:23.786123000 +0100
@@ -10,7 +10,7 @@
      EXPRESSION
       VARIABLE, ("fibonacci_number"), (nil)
       EXPRESSION_LIST
-       VARIABLE, ("n"), depth: 2, stack_offset: 8, entry label: "n"
+       VARIABLE, ("n"), (nil)
     PRINT_STATEMENT
      EXPRESSION_LIST
       CONSTANT
@@ -44,13 +44,13 @@
            VARIABLE, ("fibonacci_number"), (nil)
            EXPRESSION_LIST
             EXPRESSION
-             VARIABLE, ("n"), depth: 2, stack_offset: 8, entry label: "n"
+             VARIABLE, ("n"), (nil)
              CONSTANT
           EXPRESSION
            VARIABLE, ("fibonacci_number"), (nil)
            EXPRESSION_LIST
             EXPRESSION
-             VARIABLE, ("n"), depth: 2, stack_offset: 8, entry label: "n"
+             VARIABLE, ("n"), (nil)
              CONSTANT
     RETURN_STATEMENT
      VARIABLE, ("y"), depth: 2, stack_offset: -4, entry label: "y"
