--- correctOutput/for_simple.s9	2015-02-22 13:09:53.412145000 +0100
+++ testOutput/for_simple.s9	2015-02-22 14:33:31.693709000 +0100
@@ -1,15 +0,0 @@
- PROGRAM
- FUNCTION_LIST
-  FUNCTION, ("main"), (nil)
-   (nil)
-   STATEMENT_LIST
-    DECLARATION_STATEMENT
-    FOR_STATEMENT
-     ASSIGNMENT_STATEMENT
-      VARIABLE, ("i"), depth: 2, stack_offset: -4, entry label: "i"
-      CONSTANT
-     CONSTANT
-     STATEMENT_LIST
-      PRINT_STATEMENT
-       EXPRESSION_LIST
-        VARIABLE, ("i"), depth: 2, stack_offset: -4, entry label: "i"
