--- correctOutput/for_hard.s10	2015-02-22 13:09:53.366145000 +0100
+++ testOutput/for_hard.s10	2015-02-22 17:19:21.201838000 +0100
@@ -1,6 +1,16 @@
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression FUNC_CALL
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
