--- correctOutput/for_hard.s9	2015-02-22 13:09:53.388148000 +0100
+++ testOutput/for_hard.s9	2015-02-22 15:09:24.021146000 +0100
@@ -27,9 +27,9 @@
           EXPRESSION
            VARIABLE, ("print"), (nil)
            EXPRESSION_LIST
-            VARIABLE, ("i"), depth: 2, stack_offset: -4, entry label: "i"
-            VARIABLE, ("j"), depth: 2, stack_offset: -8, entry label: "j"
-            VARIABLE, ("k"), depth: 2, stack_offset: -12, entry label: "k"
+            VARIABLE, ("i"), (nil)
+            VARIABLE, ("j"), (nil)
+            VARIABLE, ("k"), (nil)
   FUNCTION, ("print"), (nil)
    VARIABLE_LIST
     DECLARATION_STATEMENT
