--- correctOutput/function_hard.s10	2015-02-22 13:09:53.426147000 +0100
+++ testOutput/function_hard.s10	2015-02-22 17:19:21.390840000 +0100
@@ -1,22 +1,43 @@
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking assignment
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking expression -
-Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking expression -
-Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking expression -
-Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+ASSIGNMENT_STATEMENT
