--- correctOutput/function_hard.s9	2015-02-22 13:09:53.455145000 +0100
+++ testOutput/function_hard.s9	2015-02-22 15:09:24.474133000 +0100
@@ -20,9 +20,9 @@
       EXPRESSION
        VARIABLE, ("func1"), (nil)
        EXPRESSION_LIST
-        VARIABLE, ("a"), depth: 2, stack_offset: -4, entry label: "a"
-        VARIABLE, ("b"), depth: 2, stack_offset: -8, entry label: "b"
-        VARIABLE, ("c"), depth: 2, stack_offset: -12, entry label: "c"
+        VARIABLE, ("a"), (nil)
+        VARIABLE, ("b"), (nil)
+        VARIABLE, ("c"), (nil)
   FUNCTION, ("func1"), (nil)
    VARIABLE_LIST
     DECLARATION_STATEMENT
@@ -49,10 +49,10 @@
         VARIABLE, ("func2"), (nil)
         EXPRESSION_LIST
          EXPRESSION
-          VARIABLE, ("a"), depth: 2, stack_offset: 16, entry label: "a"
+          VARIABLE, ("a"), (nil)
           CONSTANT
-         VARIABLE, ("b"), depth: 2, stack_offset: 12, entry label: "b"
-         VARIABLE, ("c"), depth: 2, stack_offset: 8, entry label: "c"
+         VARIABLE, ("b"), (nil)
+         VARIABLE, ("c"), (nil)
     RETURN_STATEMENT
      VARIABLE, ("d"), depth: 2, stack_offset: -4, entry label: "d"
   FUNCTION, ("func2"), (nil)
@@ -80,11 +80,11 @@
        EXPRESSION
         VARIABLE, ("func3"), (nil)
         EXPRESSION_LIST
-         VARIABLE, ("a"), depth: 2, stack_offset: 16, entry label: "a"
+         VARIABLE, ("a"), (nil)
          EXPRESSION
-          VARIABLE, ("b"), depth: 2, stack_offset: 12, entry label: "b"
+          VARIABLE, ("b"), (nil)
           CONSTANT
-         VARIABLE, ("c"), depth: 2, stack_offset: 8, entry label: "c"
+         VARIABLE, ("c"), (nil)
     RETURN_STATEMENT
      VARIABLE, ("d"), depth: 2, stack_offset: -4, entry label: "d"
   FUNCTION, ("func3"), (nil)
@@ -112,10 +112,10 @@
        EXPRESSION
         VARIABLE, ("func1"), (nil)
         EXPRESSION_LIST
-         VARIABLE, ("a"), depth: 2, stack_offset: 16, entry label: "a"
-         VARIABLE, ("b"), depth: 2, stack_offset: 12, entry label: "b"
+         VARIABLE, ("a"), (nil)
+         VARIABLE, ("b"), (nil)
          EXPRESSION
-          VARIABLE, ("c"), depth: 2, stack_offset: 8, entry label: "c"
+          VARIABLE, ("c"), (nil)
           CONSTANT
     RETURN_STATEMENT
      VARIABLE, ("d"), depth: 2, stack_offset: -4, entry label: "d"
