--- correctOutput/function_medium.s9	2015-02-22 13:09:53.479145000 +0100
+++ testOutput/function_medium.s9	2015-02-22 15:09:24.877137000 +0100
@@ -18,8 +18,8 @@
      EXPRESSION
       VARIABLE, ("func1"), (nil)
       EXPRESSION_LIST
-       VARIABLE, ("b"), depth: 2, stack_offset: -8, entry label: "b"
-       VARIABLE, ("d"), depth: 2, stack_offset: -16, entry label: "d"
+       VARIABLE, ("b"), (nil)
+       VARIABLE, ("d"), (nil)
        CONSTANT
        EXPRESSION
         CONSTANT
@@ -32,8 +32,8 @@
        EXPRESSION
         CONSTANT
         CONSTANT
-       VARIABLE, ("b"), depth: 2, stack_offset: -8, entry label: "b"
-       VARIABLE, ("d"), depth: 2, stack_offset: -16, entry label: "d"
+       VARIABLE, ("b"), (nil)
+       VARIABLE, ("d"), (nil)
        CONSTANT
        CONSTANT
        CONSTANT
