--- correctOutput/function_hard.s10	2015-02-22 13:09:53.426147000 +0100
+++ testOutput/function_hard.s10	2015-02-24 12:36:20.688320000 +0100
@@ -7,16 +7,13 @@
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking expression -
-Type checking expression -
 Type checking assignment
 Type checking expression >
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking expression -
-Type checking expression -
 Type checking assignment
 Type checking expression >
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking expression -
-Type checking expression -
