--- correctOutput/function_hard.s7	2015-02-22 13:09:53.450145000 +0100
+++ testOutput/function_hard.s7	2015-02-22 15:09:24.360124000 +0100
@@ -12,9 +12,6 @@
 Retrieving variable b
 Retrieving variable c
 Retrieving function func1
-Retrieving variable a
-Retrieving variable b
-Retrieving variable c
 Remove scope, index: 1 
 Add scope, index: 2 
 Inserting variable a INT depth:2 offset:16
@@ -28,9 +25,6 @@
 Retrieving variable a
 Retrieving variable d
 Retrieving function func2
-Retrieving variable a
-Retrieving variable b
-Retrieving variable c
 Retrieving variable d
 Remove scope, index: 1 
 Add scope, index: 2 
@@ -45,9 +39,6 @@
 Retrieving variable b
 Retrieving variable d
 Retrieving function func3
-Retrieving variable a
-Retrieving variable b
-Retrieving variable c
 Retrieving variable d
 Remove scope, index: 1 
 Add scope, index: 2 
@@ -62,9 +53,6 @@
 Retrieving variable c
 Retrieving variable d
 Retrieving function func1
-Retrieving variable a
-Retrieving variable b
-Retrieving variable c
 Retrieving variable d
 Remove scope, index: 1 
 Remove scope, index: 0 
