--- correctOutput/function_medium.s7	2015-02-22 13:09:53.475145000 +0100
+++ testOutput/function_medium.s7	2015-02-22 15:09:24.800127000 +0100
@@ -12,12 +12,8 @@
 Retrieving variable d
 Retrieving variable a
 Retrieving function func1
-Retrieving variable b
-Retrieving variable d
 Retrieving variable c
 Retrieving function func2
-Retrieving variable b
-Retrieving variable d
 Retrieving variable a
 Retrieving variable c
 Remove scope, index: 1 
