--- correctOutput/function_hard.s6	2015-02-22 13:09:53.448149000 +0100
+++ testOutput/function_hard.s6	2015-02-22 15:09:24.227129000 +0100
@@ -10,9 +10,6 @@
 VARIABLE: access: c
 CONSTANT
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: a
-VARIABLE: access: b
-VARIABLE: access: c
 EXPRESSION: End
 FUNCTION: End
 FUNCTION: Start: func1
@@ -31,12 +28,6 @@
 EXPRESSION: End
 VARIABLE: access: d
 EXPRESSION: Start: FUNC_CALL
-EXPRESSION: Start: -
-VARIABLE: access: a
-CONSTANT
-EXPRESSION: End
-VARIABLE: access: b
-VARIABLE: access: c
 EXPRESSION: End
 VARIABLE: access: d
 FUNCTION: End
@@ -56,12 +47,6 @@
 EXPRESSION: End
 VARIABLE: access: d
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: a
-EXPRESSION: Start: -
-VARIABLE: access: b
-CONSTANT
-EXPRESSION: End
-VARIABLE: access: c
 EXPRESSION: End
 VARIABLE: access: d
 FUNCTION: End
@@ -81,12 +66,6 @@
 EXPRESSION: End
 VARIABLE: access: d
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: a
-VARIABLE: access: b
-EXPRESSION: Start: -
-VARIABLE: access: c
-CONSTANT
-EXPRESSION: End
 EXPRESSION: End
 VARIABLE: access: d
 FUNCTION: End
