--- correctOutput/if_hard.s10	2015-02-22 13:09:53.530145000 +0100
+++ testOutput/if_hard.s10	2015-02-22 14:33:33.908719000 +0100
@@ -1,10 +0,0 @@
-Type checking assignment
-Type checking assignment
-Type checking expression UMINUS
-Type checking expression !=
-Type checking expression >=
-Type checking expression !=
-Type checking expression UMINUS
-Type checking expression >
-Type checking expression >
-Type checking expression >
