--- correctOutput/logical.s10	2015-02-22 13:09:53.588146000 +0100
+++ testOutput/logical.s10	2015-02-22 16:54:43.584898000 +0100
@@ -1,4 +1,6 @@
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking assignment
@@ -15,44 +17,94 @@
 Type checking expression ||
 Type checking assignment
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression &&
 Type checking expression &&
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression ||
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression &&
 Type checking expression ||
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ||
 Type checking expression &&
 Type checking expression !
+Type error at:
+EXPRESSION (!)
+Type error at:
+ASSIGNMENT_STATEMENT
