--- correctOutput/matrix_multiply.s10	2015-02-22 13:09:53.619150000 +0100
+++ testOutput/matrix_multiply.s10	2015-02-22 23:47:40.203202000 +0100
@@ -7,16 +7,26 @@
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression +
 Type checking expression *
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression -
 Type checking expression +
 Type checking expression *
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression NEW
 Type checking assignment
@@ -24,47 +34,95 @@
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression +
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression *
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (+)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (ARRAY_INDEX)
