--- correctOutput/merge_sort.s7	2015-02-22 13:09:53.688147000 +0100
+++ testOutput/merge_sort.s7	2015-02-22 15:09:26.134176000 +0100
@@ -12,7 +12,6 @@
 Retrieving variable i
 Retrieving variable i
 Retrieving function merge_sort
-Retrieving variable a
 Retrieving variable i
 Retrieving variable a
 Retrieving variable i
@@ -29,18 +28,8 @@
 Retrieving variable end
 Retrieving variable start
 Retrieving function merge_sort
-Retrieving variable a
-Retrieving variable start
-Retrieving variable middle
 Retrieving function merge_sort
-Retrieving variable a
-Retrieving variable middle
-Retrieving variable end
 Retrieving function merge
-Retrieving variable a
-Retrieving variable start
-Retrieving variable middle
-Retrieving variable end
 Remove scope, index: 1 
 Add scope, index: 2 
 Inserting variable a ARRAY(INT)[100] depth:2 offset:20
