--- correctOutput/merge_sort.s10	2015-02-22 13:09:53.672147000 +0100
+++ testOutput/merge_sort.s10	2015-02-23 10:37:28.438972000 +0100
@@ -1,12 +1,20 @@
 Type checking assignment
 Type checking expression NEW
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression -
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression FUNC_CALL
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression <=
 Type checking expression -
 Type checking assignment
@@ -18,35 +26,63 @@
 Type checking expression FUNC_CALL
 Type checking assignment
 Type checking expression NEW
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking assignment
 Type checking expression >=
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking assignment
 Type checking expression +
 Type checking expression >=
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking assignment
 Type checking expression +
 Type checking expression <
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
+Type error at:
+EXPRESSION (<)
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking assignment
 Type checking expression +
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking assignment
 Type checking expression +
 Type checking assignment
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (ARRAY_INDEX)
