--- correctOutput/merge_sort.s9	2015-02-22 13:09:53.693146000 +0100
+++ testOutput/merge_sort.s9	2015-02-22 15:09:26.223132000 +0100
@@ -25,7 +25,7 @@
     EXPRESSION
      VARIABLE, ("merge_sort"), (nil)
      EXPRESSION_LIST
-      VARIABLE, ("a"), depth: 2, stack_offset: -4, entry label: "a"
+      VARIABLE, ("a"), (nil)
       CONSTANT
       CONSTANT
     FOR_STATEMENT
@@ -68,22 +68,22 @@
       EXPRESSION
        VARIABLE, ("merge_sort"), (nil)
        EXPRESSION_LIST
-        VARIABLE, ("a"), depth: 2, stack_offset: 16, entry label: "a"
-        VARIABLE, ("start"), depth: 2, stack_offset: 12, entry label: "start"
-        VARIABLE, ("middle"), depth: 2, stack_offset: -4, entry label: "middle"
+        VARIABLE, ("a"), (nil)
+        VARIABLE, ("start"), (nil)
+        VARIABLE, ("middle"), (nil)
       EXPRESSION
        VARIABLE, ("merge_sort"), (nil)
        EXPRESSION_LIST
-        VARIABLE, ("a"), depth: 2, stack_offset: 16, entry label: "a"
-        VARIABLE, ("middle"), depth: 2, stack_offset: -4, entry label: "middle"
-        VARIABLE, ("end"), depth: 2, stack_offset: 8, entry label: "end"
+        VARIABLE, ("a"), (nil)
+        VARIABLE, ("middle"), (nil)
+        VARIABLE, ("end"), (nil)
       EXPRESSION
        VARIABLE, ("merge"), (nil)
        EXPRESSION_LIST
-        VARIABLE, ("a"), depth: 2, stack_offset: 16, entry label: "a"
-        VARIABLE, ("start"), depth: 2, stack_offset: 12, entry label: "start"
-        VARIABLE, ("middle"), depth: 2, stack_offset: -4, entry label: "middle"
-        VARIABLE, ("end"), depth: 2, stack_offset: 8, entry label: "end"
+        VARIABLE, ("a"), (nil)
+        VARIABLE, ("start"), (nil)
+        VARIABLE, ("middle"), (nil)
+        VARIABLE, ("end"), (nil)
   FUNCTION, ("merge"), (nil)
    VARIABLE_LIST
     DECLARATION_STATEMENT
