--- correctOutput/merge_sort.s10	2015-02-22 13:09:53.672147000 +0100
+++ testOutput/merge_sort.s10	2015-02-22 22:41:43.249734000 +0100
@@ -1,52 +1,96 @@
 Type checking assignment
 Type checking expression NEW
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression FUNC_CALL
+Type error at:
+EXPRESSION (FUNC_CALL)
+Type error at:
+EXPRESSION (FUNC_CALL)
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression <=
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (<=)
 Type checking assignment
 Type checking expression +
 Type checking expression /
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+EXPRESSION (/)
+Type error at:
+EXPRESSION (+)
 Type checking expression FUNC_CALL
 Type checking expression FUNC_CALL
 Type checking expression FUNC_CALL
 Type checking assignment
 Type checking expression NEW
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking assignment
 Type checking expression >=
+Type error at:
+EXPRESSION (>=)
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
 Type checking expression >=
+Type error at:
+EXPRESSION (>=)
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
 Type checking expression <
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (<)
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+ASSIGNMENT_STATEMENT
