--- correctOutput/merge_sort.s6	2015-02-22 13:09:53.686146000 +0100
+++ testOutput/merge_sort.s6	2015-02-22 15:09:26.051128000 +0100
@@ -17,9 +17,6 @@
 VARIABLE: access: i
 EXPRESSION: End
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: a
-CONSTANT
-CONSTANT
 EXPRESSION: End
 VARIABLE: access: i
 CONSTANT
@@ -54,20 +51,10 @@
 EXPRESSION: End
 EXPRESSION: End
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: a
-VARIABLE: access: start
-VARIABLE: access: middle
 EXPRESSION: End
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: a
-VARIABLE: access: middle
-VARIABLE: access: end
 EXPRESSION: End
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: a
-VARIABLE: access: start
-VARIABLE: access: middle
-VARIABLE: access: end
 EXPRESSION: End
 FUNCTION: End
 FUNCTION: Start: merge
