--- correctOutput/newton.s9	2015-02-22 13:09:53.737148000 +0100
+++ testOutput/newton.s9	2015-02-22 15:09:26.703131000 +0100
@@ -10,7 +10,7 @@
      EXPRESSION
       VARIABLE, ("improve"), (nil)
       EXPRESSION_LIST
-       VARIABLE, ("n"), depth: 2, stack_offset: 8, entry label: "n"
+       VARIABLE, ("n"), (nil)
        CONSTANT
     PRINT_STATEMENT
      EXPRESSION_LIST
@@ -48,8 +48,8 @@
        EXPRESSION
         VARIABLE, ("improve"), (nil)
         EXPRESSION_LIST
-         VARIABLE, ("n"), depth: 2, stack_offset: 12, entry label: "n"
-         VARIABLE, ("next"), depth: 2, stack_offset: -4, entry label: "next"
+         VARIABLE, ("n"), (nil)
+         VARIABLE, ("next"), (nil)
      STATEMENT_LIST
       RETURN_STATEMENT
        EXPRESSION
