--- correctOutput/newton.s6	2015-02-22 13:09:53.731147000 +0100
+++ testOutput/newton.s6	2015-02-22 15:09:26.585139000 +0100
@@ -4,8 +4,6 @@
 DECLARATION: parameter/variable : 'square_root', offset: -4
 VARIABLE: access: square_root
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: n
-CONSTANT
 EXPRESSION: End
 CONSTANT
 VARIABLE: access: n
@@ -39,8 +37,6 @@
 VARIABLE: access: estimate
 EXPRESSION: End
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: n
-VARIABLE: access: next
 EXPRESSION: End
 EXPRESSION: Start: -
 VARIABLE: access: next
