--- correctOutput/newton.s8	2015-02-22 13:09:53.735147000 +0100
+++ testOutput/newton.s8	2015-02-22 14:57:30.409329000 +0100
@@ -1,16 +0,0 @@
-.syntax unified
-.cpu cortex-a15
-.fpu vfpv3-d16
-.data
-.align	2
-.DEBUG: .ascii "Hit Debug\n\000"
-.DEBUGINT: .ascii "Hit Debug, r0 was: %d\n\000"
-.INTEGER: .ascii "%d \000"
-.FLOAT: .ascii "%f \000"
-.NEWLINE: .ascii "\n \000"
-.STRING0: .ascii "The square root of"
-.ascii "\000"
-.STRING1: .ascii "is"
-.ascii "\000"
-.globl main
-.align	2
