--- correctOutput/sorting.s9	2015-02-22 13:09:53.824147000 +0100
+++ testOutput/sorting.s9	2015-02-22 15:09:27.195152000 +0100
@@ -28,7 +28,7 @@
      EXPRESSION
       VARIABLE, ("sort"), (nil)
       EXPRESSION_LIST
-       VARIABLE, ("a"), depth: 2, stack_offset: -4, entry label: "a"
+       VARIABLE, ("a"), (nil)
     FOR_STATEMENT
      ASSIGNMENT_STATEMENT
       VARIABLE, ("i"), depth: 2, stack_offset: -12, entry label: "i"
