--- correctOutput/sorting.s10	2015-02-22 13:09:53.796147000 +0100
+++ testOutput/sorting.s10	2015-02-22 15:53:52.678308000 +0100
@@ -1,22 +1,12 @@
 Type checking assignment
-Type checking expression NEW
 Type checking assignment
 Type checking assignment
-Type checking expression ARRAY_INDEX
-Type checking expression -
 Type checking assignment
-Type checking expression FUNC_CALL
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking assignment
 Type checking assignment
 Type checking expression <
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
 Type checking assignment
-Type checking expression ARRAY_INDEX
 Type checking assignment
-Type checking expression ARRAY_INDEX
-Type checking expression ARRAY_INDEX
 Type checking assignment
-Type checking expression ARRAY_INDEX
