--- correctOutput/sorting.s10	2015-02-22 13:09:53.796147000 +0100
+++ testOutput/sorting.s10	2015-02-22 22:38:35.572770000 +0100
@@ -1,9 +1,15 @@
 Type checking assignment
 Type checking expression NEW
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression FUNC_CALL
 Type checking assignment
@@ -13,6 +19,8 @@
 Type checking expression <
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+EXPRESSION (<)
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking assignment
