--- correctOutput/sorting.s10	2015-02-22 13:09:53.796147000 +0100
+++ testOutput/sorting.s10	2015-02-22 17:07:53.034136000 +0100
@@ -1,22 +1,44 @@
 Type checking assignment
 Type checking expression NEW
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression -
+Type error at:
+EXPRESSION (-)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression FUNC_CALL
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression ARRAY_INDEX
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression <
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ARRAY_INDEX
 Type checking expression ARRAY_INDEX
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking assignment
 Type checking expression ARRAY_INDEX
+Type error at:
+ASSIGNMENT_STATEMENT
