--- correctOutput/strings_and_comments.s7	2015-02-22 13:09:53.839148000 +0100
+++ testOutput/strings_and_comments.s7	2015-02-22 14:16:52.643418000 +0100
@@ -1,10 +0,0 @@
-Add scope, index: 0 
-Add scope, index: 1 
-Inserting function main INT<=() 
-Add scope, index: 2 
-Add strings ("Hello, world!"), index: 0 
-Add strings (""), index: 1 
-Add strings ("Hello, \"world\"!"), index: 2 
-Add strings ("//Comment"), index: 3 
-Remove scope, index: 1 
-Remove scope, index: 0 
