--- correctOutput/type_errors_functions.s8	2015-02-22 13:09:53.925147000 +0100
+++ testOutput/type_errors_functions.s8	2015-02-22 14:48:59.512625000 +0100
@@ -1,12 +0,0 @@
-.syntax unified
-.cpu cortex-a15
-.fpu vfpv3-d16
-.data
-.align	2
-.DEBUG: .ascii "Hit Debug\n\000"
-.DEBUGINT: .ascii "Hit Debug, r0 was: %d\n\000"
-.INTEGER: .ascii "%d \000"
-.FLOAT: .ascii "%f \000"
-.NEWLINE: .ascii "\n \000"
-.globl main
-.align	2
