--- correctOutput/type_errors_functions.s9	2015-02-22 13:09:53.927148000 +0100
+++ testOutput/type_errors_functions.s9	2015-02-22 15:09:27.880130000 +0100
@@ -18,20 +18,20 @@
     EXPRESSION
      VARIABLE, ("two_args"), (nil)
      EXPRESSION_LIST
-      VARIABLE, ("a"), depth: 2, stack_offset: -4, entry label: "a"
+      VARIABLE, ("a"), (nil)
     EXPRESSION
      VARIABLE, ("two_args"), (nil)
      EXPRESSION_LIST
-      VARIABLE, ("b"), depth: 2, stack_offset: -8, entry label: "b"
-      VARIABLE, ("a"), depth: 2, stack_offset: -4, entry label: "a"
+      VARIABLE, ("b"), (nil)
+      VARIABLE, ("a"), (nil)
     EXPRESSION
      VARIABLE, ("two_args"), (nil)
      (nil)
     EXPRESSION
      VARIABLE, ("void_func"), (nil)
      EXPRESSION_LIST
-      VARIABLE, ("a"), depth: 2, stack_offset: -4, entry label: "a"
-      VARIABLE, ("b"), depth: 2, stack_offset: -8, entry label: "b"
+      VARIABLE, ("a"), (nil)
+      VARIABLE, ("b"), (nil)
   FUNCTION, ("void_func"), (nil)
    (nil)
    STATEMENT_LIST
