--- correctOutput/type_errors_functions.s7	2015-02-22 13:09:53.923148000 +0100
+++ testOutput/type_errors_functions.s7	2015-02-22 15:09:27.822133000 +0100
@@ -12,14 +12,9 @@
 Retrieving variable a
 Retrieving function bool_func
 Retrieving function two_args
-Retrieving variable a
 Retrieving function two_args
-Retrieving variable b
-Retrieving variable a
 Retrieving function two_args
 Retrieving function void_func
-Retrieving variable a
-Retrieving variable b
 Remove scope, index: 1 
 Add scope, index: 2 
 Remove scope, index: 1 
