--- correctOutput/type_errors_assignment.s10	2015-02-22 13:09:53.877147000 +0100
+++ testOutput/type_errors_assignment.s10	2015-02-22 16:59:38.261476000 +0100
@@ -2,17 +2,7 @@
 Type error at:
 ASSIGNMENT_STATEMENT
 Type checking assignment
-Type error at:
-ASSIGNMENT_STATEMENT
 Type checking assignment
-Type error at:
-ASSIGNMENT_STATEMENT
 Type checking assignment
-Type error at:
-ASSIGNMENT_STATEMENT
 Type checking assignment
-Type error at:
-ASSIGNMENT_STATEMENT
 Type checking assignment
-Type error at:
-ASSIGNMENT_STATEMENT
