--- correctOutput/type_errors_functions.s6	2015-02-22 13:09:53.921148000 +0100
+++ testOutput/type_errors_functions.s6	2015-02-22 15:09:27.768130000 +0100
@@ -9,17 +9,12 @@
 EXPRESSION: Start: FUNC_CALL
 EXPRESSION: End
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: a
 EXPRESSION: End
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: b
-VARIABLE: access: a
 EXPRESSION: End
 EXPRESSION: Start: FUNC_CALL
 EXPRESSION: End
 EXPRESSION: Start: FUNC_CALL
-VARIABLE: access: a
-VARIABLE: access: b
 EXPRESSION: End
 FUNCTION: End
 FUNCTION: Start: void_func
