--- correctOutput/while_simple.s6	2015-02-22 13:09:53.996152000 +0100
+++ testOutput/while_simple.s6	2015-02-22 14:33:42.129763000 +0100
@@ -1,17 +1,3 @@
 FUNCTION_LIST: Start
 FUNCTION: Start: main
-DECLARATION: parameter/variable : 'a', offset: -4
 VARIABLE: access: a
-CONSTANT
-EXPRESSION: Start: >
-VARIABLE: access: a
-CONSTANT
-EXPRESSION: End
-VARIABLE: access: a
-VARIABLE: access: a
-EXPRESSION: Start: -
-VARIABLE: access: a
-CONSTANT
-EXPRESSION: End
-FUNCTION: End
-FUNCTION_LIST: End
