--- correctOutput/while_hard.s10	2015-02-22 13:09:53.957148000 +0100
+++ testOutput/while_hard.s10	2015-02-22 14:48:59.893627000 +0100
@@ -1,15 +0,0 @@
-Type checking assignment
-Type checking expression >
-Type checking assignment
-Type checking expression >
-Type checking assignment
-Type checking expression UMINUS
-Type checking expression <
-Type checking expression *
-Type checking expression *
-Type checking assignment
-Type checking expression +
-Type checking assignment
-Type checking expression -
-Type checking assignment
-Type checking expression -
