--- correctOutput/while_hard.s10	2015-02-22 13:09:53.957148000 +0100
+++ testOutput/while_hard.s10	2015-02-22 22:29:08.585857000 +0100
@@ -1,15 +1,35 @@
 Type checking assignment
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking assignment
 Type checking expression >
+Type error at:
+EXPRESSION (>)
 Type checking assignment
 Type checking expression UMINUS
+Type error at:
+EXPRESSION (UMINUS)
+Type error at:
+ASSIGNMENT_STATEMENT
 Type checking expression <
+Type error at:
+EXPRESSION (<)
 Type checking expression *
 Type checking expression *
+Type error at:
+EXPRESSION (*)
+Type error at:
+EXPRESSION (*)
 Type checking assignment
 Type checking expression +
+Type error at:
+EXPRESSION (+)
 Type checking assignment
 Type checking expression -
+Type error at:
+EXPRESSION (-)
 Type checking assignment
 Type checking expression -
+Type error at:
+EXPRESSION (-)
